% ****** Start of file sorsamp.tex ******
%
%   This file is part of the AIP files in the AIP distribution for REVTeX 4.
%   Version 4.2a of REVTeX, December 2014
%
%   Copyright (c) 2014 American Institute of Physics.
%
%   See the AIP README file for restrictions and more information.
%
% TeX'ing this file requires that you have AMS-LaTeX 2.0 installed
% as well as the rest of the prerequisites for REVTeX 4.2
%
% It also requires running BibTeX. The commands are as follows:
%
%  1)  latex  sorsamp
%  2)  bibtex sorsamp
%  3)  latex  sorsamp
%  4)  latex  sorsamp
%
% Use this file as a source of example code for your aip document.
% Use the file aiptemplate.tex as a template for your document.
\documentclass[%
 sor,
%aip,
%twoside,
%groupedaddress,
%jmp,
 jor,
 amsmath,amssymb,
%preprint,%
 reprint,%
%author-year,%
%author-numerical,%
]{revtex4-2}

\usepackage{graphicx}% Include figure files
\usepackage{dcolumn}% Align table columns on decimal point
\usepackage{bm}% bold math
\usepackage{hyperref}% add hypertext capabilities
%\usepackage[mathlines]{lineno}% Enable numbering of text and display math
%\linenumbers\relax % Commence numbering lines

% \renewcommand{\emph}{\textsc}   % emphasize with small caps
% \renewcommand{\emph}{\textbf}
% \renewcommand{\emph}{\textit{\textbf}} 


\begin{document}

\preprint{AIP/123-QED}

\title[Draft]{Title TBD}% Force line breaks with \\
\thanks{Contact:}

\author{Shao-Kai Jonathan Huang, Wei-Hsiang Lin}
\affiliation{%
  Department of Physics, National Taiwan University, Taipei 10617 \\
  Institute of Molecular Biology, Academia Sinica, Taipei 115
}%
\altaffiliation[Also at ]{}%Lines break automatically or can be forced with \\

\date{\today}% It is always \today, today,
             %  but any date may be explicitly specified

\begin{abstract}
    This study develops a mathematical model of proteome partition in bacterial cells to study their allocation strategies, focusing on ribosomal content and its impact on growth rate. We study scalable reaction networks (SNRs) and formulate a generalized multi-sector partition model. We show analytically how nutrient influx and bottleneck reactions affect resource allocation strategy and thus cell growth, and derive novel analytic solutions for various limiting cases. Simulations reveal that the almost-linear correlation between growth rate and ribosomal protein content emerge out of the system dynamics. The findings offer important insights into the mathematical theory of cellular resource allocation.
\end{abstract}

\keywords{Proteome partition, cellular resource allocation, reaction networks, metabolic engineering}%Use showkeys class option if keyword
                              %display desired
\maketitle


% Literature Review should be included in the Introduction section
% \section{\label{sec:review}Literature Review}

\section{\label{sec:introduction}Introduction}
Understanding how cells grow is a fundamental problem in biology. The phenomenon of growth emerges from many complex interactions within the cellular system. Surprisingly, however, it can be quantitativly studied by looking at simple emergent growth laws. At the molecular level, growth is limited primarily by the rate of protein synthesis, which is controlled by ribosomes. It is a well-known fact that ribosomes are \emph{autocatalytic machines}, as they synthesize essentially all cellular proteins, including the ribosomal proteins and RNA polymerases that constitute the translation machinery itself. Thus, bacterial growth can naturally be viewed as the dynamics of an autocatalytic reaction network in which the catalysts continuously maintain themselves while driving biomass production. Studying cell growth from a reaction-network perspective in both the Eulerian and Lagrangian view, as inspired by fluid mechanics, allows us to ask how evolution has tuned these autocatalytic systems to optimize growth.

\subsection{\label{ssec:snr}Scalable Reaction Networks}
It is observed that during the exponential phase, all components of a bacteria cell grows exponentially with the same rate \cite{cooper1991bacterial}. This is known as the \textbf{balanced growth condition}, and is essential in order to maintain exponential growth. Because of this, the number and biomass density of quantities remain nearly invariant during exponential growth. More recently, \emph{flux-allocation models} in \cite{Chure_eLife} showed that optimal allocation arises when translational flux and metabolic flux balance each other. Under balanced growth, cell density remain approximately constant as demonstrated by experiments, and the cellular system may be modeled as a \emph{scalable reaction network} (SRN) as defined (\cite{Lin2020}).

Michaelis-Menten kinetics can be derived from the assumption that substrate level is sufficiently high, and we will assume the system fluxes to be of the form (\ref{equ:MM}). This is also the form of general growth rate, per Monod. 
\begin{equation}
    \frac{\mathrm{d}[C]}{\mathrm{d}t} = \frac{V_\text{M} [C]}{K_\text{M} + [C]}. \label{equ:MM}
\end{equation} 


\subsection{Proteome Partition and Bacterial Growth Laws}

\subsubsection{Balanced Growth}
In nutrient-replete conditions, bacteria such as \textit{Escherichia coli} can grow and divide at an exponential rate. This is known as \emph{exponential growth}. At the same time, the size of the bacteria also increases exponentially, leading to a constant concentration of cellular components. For a system at equilibrium, the fluxes of biomass must obey the \emph{principle of detailed balance}, where each forward and reverse reaction rates are equal. However, detailed balanced is a too stringent condition, and moreover it is known that living things operate far from equilibrium, and thus do not satisfy detailed balance. Instead, during exponential growth, the fluxes of biomass satisfy a condition called \emph{balanced growth}, where the biomass concentrations remain constant even as the total biomass increases exponentially. That is, it resides in the steady state of biomass concentration.

\subsubsection{The Bacterial Proteome}
The \emph{proteome} is the complete set of proteins expressed by an organism. Unlike the genome, the proteome composition changes actively and rapidly in response to environmental conditions and internal demands. In \textit{E. coli}, the proteome consists of more than 1000 distinct types of proteins, and hence a coarse-grained \emph{proteome-allocation model} of the proteome dynamics is often introduced to study their dynamics. A \emph{proteome partition} is such a scheme that is often based on biological functions of the proteins and defined by the protein biomass fraction in each sector.

A series of experiments over the past decades has revealed surprisingly simple quantitative relationships, called \emph{bacterial growth laws}, linking growth rate to global proteome allocation. In particular, measurements in \textit{E.~coli} showed that the fraction of the proteome devoted to translation increases linearly with the growth rate (\cite{Scott2010}). These observations suggest a coordinated, global allocation of proteome sectors. Understanding the relevant mechanisms is crucial for elucidating bacterial physiology, and has implications for both bioengineering and medicine.

\subsubsection{Classic Model of Proteome Partition and Growth Laws}
\cite{Scott2010} proposed a three-sector proteome partition model to explain linear relationships arising in bacteria growth laws. The model consists of a transporter sector, a metabolic sector, and a ribosomal sector, each described respectively by a number $ P_1, P_2, P_3 $ in between $ 0 $ and $ 1 $, such that $ P_1 + P_2 + P_3 = 1 $. These numbers are called the \emph{proteome fraction}, and together define the \emph{proteome partition} $ (P_1, P_2, P_3) $. The transporter sector is responsible for nutrient uptake; the metabolic sector is responsible for converting metabolites into protein precursors; while the ribosomal sector makes up the translation machinery and is directly responsible for protein synthesis. We may assume an SNR model with Michaelis-Menten-type fluxes, and hence the \emph{long term growth rate} 
\begin{equation}
    \lambda = \lim_{t \to \infty} \frac{1}{t} \ln \frac{N(t)}{N(0)}
\end{equation}
must exist (\cite{Lin2020}), allowing for rigorous analysis. Experimentally, it is found that under nutrient quality modulation or translational inhibition, the growth rate $ \lambda $ exhibits two linear relationships with respect to the ribosomal fraction $ P_3 $. They are respectively described phenomenologically by 
\begin{equation}
    P_3 = P_3^{(0)} + \frac{\lambda}{\kappa_t}, \quad P_3 = P_3^{\,\text{max}} - \frac{\lambda}{\kappa_n},
\end{equation}
where $ \kappa_t $ is called the \emph{translational capacity} and is proportional to the speed of translational elongation, and $ \kappa_n $ is called the \emph{nutritional capacity}. The linear correlation is expected if the ribosomes are growth-limiting and engaged in translation at a constant rate. These relationships are called the \emph{bacterial growth laws}, and have been experimentally verified in \textit{E.~coli} and other bacteria. This landmark result is summarized in Figure~\ref{fig:growth_laws}.
\begin{figure}
    \centering
    \includegraphics[width=0.6\textwidth]{Images/growth_laws.png}%
    \caption{Bacterial growth laws relating growth rate $ \lambda $ to ribosomal translation fraction $ \theta_3 $ under different conditions. It will later be shown that this is equivalent to the proteome fraction $ P_3 $. (a) When nutrient quality is varied, $ \lambda $ is linearly correlated with $ P_3 $. (b) When translation is inhibited by antibiotics, $ \lambda $ is linearly anticorrelated with $ P_3 $.}
    \label{fig:growth_laws}
\end{figure}

\subsection{Starvation and Excessive Nutrient}
Nutrient availability profoundly shapes the physiology and proteome allocation of Escherichia coli. Under nutrient-limited conditions, such as carbon or nitrogen starvation, E. coli cells reduce growth rate to prioritize survival over proliferation. This includes upregulation of proteins involved in nutrient transport, while reducing the biosynthesis of ribosomes \cite{Scott2010}\cite{Hui}. The stringent response, mediated by alarmone molecules like (p)ppGpp, plays a key role in reallocating proteomic resources decreasing ribosomal synthesis under starvation \cite{Potrykus2008ppGpp}. Conversely, when nutrients are abundant, E. coli cells exhibit acceleratted growth and allocate a larger fraction of the proteome to ribosomal proteins. Understanding how E. coli balances proteome partitioning across nutrient extremes is essential for elucidating fundamental principles of resource optimization.

\subsection{Overview of Results} 

Despite the rich literature on proteome partition and bacteria growth physiology, the current studies are largely phenomenological. As a result, it is often unclear how proteome allocation emerge dynamically from the autocatalytic reaction network that constitutes the cell. In this work, we develop a theory of proteome partition with SNRs, and find that growth laws emerge from the network dynamics without external assumptions. Then, we analyze how proteome allocation depend on nutrient availability, and in particular elucidate the behavior of regimes of severe nutrient scarcity and abundance. In some regimes, the theory yields tractable analytic expressions. 

A discussion of the experimental and theoretical literature is in Section~\ref{sec:introduction}. Details of the model are introduced in SI Section (). Subsequent sections analyze the system dynamics, explore the starvation and nutrient overabundance regimes, and present analytical results for the generalized proteome partition model.

% Theory 

\section{\label{sec:theory}Theory}

\subsection{\label{ssec:two_sector}A Simplified Model Recovers Known Analytic Result}
Before analyzing dynamics of the classic three-sector model, we first consider a minimalistic mass action model of only three nodes, corresponding to a two-sector proteome partition. This model consists of a nutrient uptake reaction, a metabolic reaction, and a translation reaction. By SI Section (), the steady states are 
\begin{equation}
    Y_{1*} = \left(1 + \frac{r \theta_2}{b \theta_1}\right)^{-1}, \quad Y_{2*} = \frac{r \theta_2}{b} Y_{1*} , \quad Y_{3*} = \frac{r \theta_2^2}{b \theta_1} Y_{1*}.
\end{equation}

What is great about this simple model is that the optimal proteome partition can be solved analytically. By SI Section (), the optimal proteome partition is
\begin{equation}
    \theta_{1*} = \frac{1}{1 + \sqrt{b/r}}, \quad \theta_{2*} = \frac{\sqrt{b/r}}{1 + \sqrt{b/r}}, \quad \text{for all } b > 0.
\end{equation}

The relationship between growth rate $ \lambda $ and optimal "ribosome biomass fraction" $ Y_{3*} $ is found to be exactly equation (\ref{equ:two_sector_growth_law}). The square root followed by a linear term has been derived by other means in the literature, albeit with more assumptions (\cite{Dourado2020AnalyticalBalancedGrowth}).
\begin{subequations}
    \label{equ:two_sector_growth_law}
    \begin{align}
        \lambda &= b Y_2 = \frac{b}{\left(1+\sqrt{b/r}\right)^2}, \\
        Y_{3*} &= Y_{3*} (\lambda) = \sqrt{\frac{\lambda}{r}} - \frac{\lambda}{r}.
    \end{align}
\end{subequations}
Here the ribosomal protein fraction $ Y_{3*} $ has closed-form relationship with $ \lambda $, while $ \lambda $ is not a function of $ Y_{3*} $. There is a maximum ribosomal biomass fraction which does not correspond to the maximum growth rate. However, if we attempt a similar analysis for its three-sector analogue or introduce Michaelis-Menten kinetics into the current model, optimization becomes impossible to do analytically. This motivates tour main results: introducing approximation schemes to express protein allocation in approximate closed form.

\subsection{\label{ssec:three_sector}Three-Sector Partition Model}

\begin{figure}
    \centering
    \includegraphics[width=0.8\linewidth]{Images/general_model.png}
    \caption{Diagram of the nonlinear reaction network associated with the proteome partition model.}
    \label{fig:n_sector}
\end{figure}

Consider the model with 5 nodes and 5 fluxes, corresponding to the $n=3$ case in Fig~\ref{fig:n_sector}. Each flux obeys Michaelis-Menten kinetics. The associated system of ODE obeyed by the \textit{biomass fraction} $ Y = \left(Y_1, Y_2, Y_3, Y_4, Y_5 \right) $ is 
\begin{align}
    \frac{\mathrm{d}Y_1}{\mathrm{d}t} &= bY_3 - \frac{a_1 Y_1 Y_4}{k_1 + Y_1} - bY_3Y_1, \\
    \frac{\mathrm{d}Y_2}{\mathrm{d}t} &= \frac{a_1 Y_1 Y_4}{k_1 + Y_1} - \frac{a_1 Y_2 Y_5}{k_2 + Y_2} - bY_3Y_2, \\
    \frac{\mathrm{d}Y_k}{\mathrm{d}t} &= \theta_k \frac{a_1 Y_2 Y_5}{k_2 + Y_2} \; (k=1,2,3).
\end{align}

Intuitive, we would expect that exchanging two fluxes in the metabolic chain would not affect the growth rate. Below we will see that this is indeed the case. Given a proteome partition $ \theta $, the steady state $ Y^* = Y^*[\theta_1, \theta_2, \theta_3] $ can be explicitly written as equation (\ref{equ:Y_steady_sol}), and we denote the $ \theta $-optimal steady state as $ Y_{\ast} = Y[\theta_{\ast}] $, where the optimal partition $ \theta_{\ast} $ will be determined later.  

\begin{equation*}
    1 - Y_1^* = \frac{1}{2} \left[ -\sqrt{\left(\frac{a_1 \theta_2}{b \theta_1}+ k_1 - 1\right)^{2} + 4k_1} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 + 1\right) \right]. 
\end{equation*}

% \notag disables numbering
% This is the analytic solution of the system
\begin{subequations}
    \label{equ:Y_steady_sol}
    \begin{align}
        %
        Y_1^* &= \frac{1}{2} \left[ \sqrt{\left(\frac{a_1 \theta_2}{b \theta_1} + k_1 - 1\right)^{2} + 4k_1} - \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 - 1\right) \right], \\
        %
        Y_2^* &= \frac{1}{2} \left[ \sqrt{\left(\frac{a_2 \theta_3}{b \theta_1} + k_2 - (1 - Y_1^* )\right)^{2} + 4 (1 - Y_1^*) k_2} - \left(\frac{a_1 \theta_2}{b \theta_1} + k_2 - (1 - Y_1^* ) \right) \right] \notag \\ 
        &= \frac{1}{2} \sqrt{
            \begin{aligned}
                & \left\{ \frac{a_2 \theta_3}{b \theta_1} + k_2 - \frac{1}{2} \left[ -\sqrt{\left(\frac{a_1 \theta_2}{b \theta_1}+ k_1 - 1\right)^{2} + 4k_1} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 + 1\right) \right] \right\}^{2} + \\
                &4k_1 \left\{ \frac{1}{2} \left[ -\sqrt{\left(\frac{a_1 \theta_2}{b \theta_1}+ k_1 - 1\right)^{2} + 4k_1} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 + 1\right) \right] \right\} \\
            \end{aligned}
        } \notag \\ 
        & - \frac{1}{2} \left\{ \frac{a_2 \theta_3}{b \theta_1} + k_2 - \frac{1}{2} \left[ - \sqrt{ \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 -1\right)^{2} + 4 k_1} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 + 1\right) \right] \right\}, \\
        %
        Y_3^* &= \left( \frac{a_2 \theta_3}{b}\right) \left(\frac{Y_2^*}{k_2 + Y_2^*}\right), \\
        %
        Y_4^* &= \left( \frac{a_2 \theta_2 \theta_3}{b \theta_1}\right) \left(\frac{Y_2^*}{k_2 + Y_2^*}\right), \\
        %
        Y_5^* &= \left( \frac{a_2 \theta_3^{2}}{b \theta_1}\right) \left(\frac{Y_2^*}{k_2 + Y_2^*}\right), 
    \end{align}
\end{subequations}

\subsection{\label{ssec:fitness}Optimizing Over the Fitness Landscape}
In the context of ecology, the optimization objetcive is long-term \emph{fitness}, which does not necessarily correspond to fastest growth, although \emph{for non-interacting unicellular organisms in constant environments, fitness is equivalent to growth rate} (\cite{Bruggeman2023}, \cite{Dourado2020AnalyticalBalancedGrowth}). For two asexual strains in a constant environment with different exponential growth rates, the strain with higher Malthusian parameter $\lambda$ takes over exponentially. In this example, growth rate is equal to fitness. However, in some cases maximizing growth rate does not lead to the evolutionary outcome, and our model results do not apply. Fist, yield may be favored over growth rate in nutrient-poor or closed systems, because the population that turns limited resources into the largest final population can leave more descendants (\cite{Lipson}, \cite{Abram2021GrowthSurvivalTradeoffs}). This underpins the \emph{copiotroph-oligotroph dichotomy}. Second, in fluctuating environments, the genotype with slower growth but much better stress response can have higher long-term fitness (\cite{Zhu2024Shaping}), while phenotypes with non-optimal growth rate have been observed to respond to antibiotics better (\cite{Kratz2024}). Finally, bet-hedging, where organisms suffer decreased fitness in their typical conditions in exchange for increased fitness in stressful conditions, requires slower or heterogeneous growth (\cite{Muntoni2022FitnessHeterogeneity}), while a faster growing phenotype may be subject to the \emph{Kill-the-Winner hypothesis} (\cite{Thingstad1997}).


\subsection{Cell Growth Under Starvation}
During starvation, the nutrient level is reduced, so that $b/a_1, b/a_2 \ll 1$. We can expand the quantities in analytic form as a perturbative series in $b$ (which turns out to be equivalent to an expansion in $b\theta_1$ or $b/a_1, b/a_2$) when nutrient level is low, for details see SI section 2B. Define the variables $A = \sqrt{\frac{k_1}{a_1}} + \sqrt{\frac{k_2}{a_2}}, B = \frac{k_1}{a_1} + \frac{k_2}{a_2}, C = \sqrt{\frac{k_1k_2}{a_1 a_2}}, D = \frac{1}{a_1}+\frac{1}{a_2}$, we obtain a novel analytic expression for the partition fractions to leading order: 
\begin{subequations}
    \begin{align}
        \theta_1 &= 1 - A\sqrt{b} - \frac{1}{2}A^3 b^{3/2},\\
        \theta_2 &= \sqrt{\frac{k_1}{a_1}} \left[\sqrt{b} - \frac{1}{2}A^2 b^{3/2} \right], \\
        \theta_3 &= \sqrt{\frac{k_2}{a_2}} \left[\sqrt{b} - \frac{1}{2}A^2 b^{3/2} \right].
    \end{align} 
\end{subequations}

Notice that $\theta_1 = O(1)$, $\theta_2, \theta_3 = O(\sqrt{b})$, and this equation characterize the main finding: squareroot dependence of partition on nutrient level. Using the above, let $R = Y_{5*}$ be the ribosomal protein fraction and $\text{Small} = Y_{1*} + Y_{2*}, \text{Large} = Y_{3*}+Y_{4*}+Y_{5*}$ be the fraction of small and large molecules, respectively. Then
\begin{align}
    R &= \sqrt{\frac{k_2}{a_2}}\left[\sqrt{b} - Ab + \left(\frac{1}{2}A^2 + B + C - D\right)b^{3/2}\right], \\
    \text{Small} &= A\sqrt{b} - \left(A^2+B+C-D\right)b, \quad \text{Large} = 1 - \text{Small}.
\end{align}
The small molecule fraction is invariant under exchange of indices, and since $1$ is invariant, the large molecule fraction is also invariant. Finally, the growth rate is
\begin{equation}
    \lambda = b - 2Ab^{3/2} + \left[ \frac{5}{2} A^2 + \frac{1}{2} B - D -E\right]b^2, 
\end{equation}
which is \emph{quadratic} in ribosomal content in the leading order. In the region of intermediate nutrient level, we recover the linear relationship as observed in experiments. 
\begin{equation}
    R (\lambda) = \sqrt{\frac{k_2}{a_2}}\;\sqrt{\lambda}\left[ 1 + \frac{1}{2}\bigl(A^2 - C - D\bigr)\lambda \right].
\end{equation}

\subsection{Cell Growth Under Nutrient Overabundance}
Now, it is natural to consider the opposite limit, in whichl high levels of nutrients is supplied to the cell. This section will analyze the relevant dependencies.

\subsubsection{Michaelis-Menten Analysis}
Set $P = \theta_3/\theta_1 b, Q = \theta_2/\theta_1 b$. Numerical experiments with $a_1 = 23.8, a_2 = 1.42, k_1 = 0.1, k_2 = 0.003$ give the values $P \approx 0.350, Q \approx 0.007$, so $P \ll 1, Q \ll 1$ is justified. 

Expanding to leading order in $b^{-1}$ gives $\lambda
(\theta) = \frac{Z}{b}\left(\frac{\theta_2 \theta_3}{\theta_1}\right)$, $Z = a_1a_2/[(1+k_1)k_2]$. Observe that the maximum of $\frac{\theta_2\theta_3}{\theta_1}$ occurs on the boundary, where $\theta_1 \to 0$, and consequently $\theta_2 = \theta_3 = \frac{1}{2}$ by the AM-GM inequality. The correct prediction should see $\theta_3 \to 1$, $\theta_2 \approx 0$, and $\theta_1 \to 0$, so an expansion to $O(b^{-2})$ is sought. Now the approximate theory becomes too cumbersome to yield a clean solution, though one has been explicitly derived in SI Section (). Hence, the low metabolite fraction approximation comes in handy, which will be discussed in the following section. 

\subsubsection{Reaction Under Low Metabolite Fractions}
Proteins in nature have evolved to have small values of $ k $, and thus most reactions happen in strictly the Michaelis-Menten regime. However, for extremely low metabolite fractions, reactions are well-approximated by mass action kinetics. In this case, we may identify $ r_i $ with $ a_i / k_i $, and write the steady states of the system as 
\begin{subequations}
    \begin{align}
        Y_1^* &= \left( 1 + \frac{r_1 \theta_{2}}{b \theta_{1}} \right)^{-1}, \\
        Y_2^* &= \left(\frac{r_1 \theta_{2}}{b \theta_{1}}\right) \left[ \left(1 + \frac{r_2 \theta_{3}}{b \theta_{1}} \right)\left( 1 + \frac{r_1 \theta_{2}}{b \theta_{1}} \right) \right] ^{-1}, \\
        Y_3^* &= \left(\frac{r_{2}\theta_{3}}{b}Y_2^* \right), \quad Y_4^* = \left(\frac{r_2 \theta_2 \theta_3 }{b \theta_1} \right) Y_2^*, \quad Y_5^* = \left(\frac{r_2 \theta_3^2 }{b \theta_1} \right) Y_2^*.
    \end{align}
\end{subequations}

Carrying out a similar analysis in the SI Section (), novel analytical expressions for the optimal partition fractions under mass action kinetics are derived in the SI Section (). Importanly, $ \theta_1 \to 0 $ during dtarvation, and the ratio between the metabolic and ribosomal partitions 
\begin{equation}
    \frac{\theta_3}{\theta_2} \sim \frac{r_1}{r_2} \left(\frac{2r_1 + r_2 - \sqrt{r_2^2 + 4r_1 r_2}}{-r_2 + \sqrt{r_2^2 + 4r_1 r_2}}\right). 
\end{equation}

When the two steps are described by similar kinetics, i.e., $ r_1 = r_2 = r $, such as when a biochemical reaction is carried out twice in succession, the optimal partition is simply
\begin{subequations}
    \begin{align}
        \theta_1 &= \frac{2r}{b} + \frac{2r^2 (2 + \varphi)}{b^2} + O\left(b^{-3}\right), \\
        \theta_2 &= \frac{1}{\varphi^2} - \left(\frac{4 \varphi}{1+ 3 \varphi}\right) \frac{r}{b} + O\left(b^{-2}\right), \\
        \theta_3 &= \frac{1}{\varphi} - \left(\frac{2 + 2\varphi}{1 + 3 \varphi}\right) \frac{r}{b} + O\left(b^{-2}\right).
    \end{align}
\end{subequations}
Notice that $ b \theta_1 = O(1) $, and hence nutrient influx is constant and well-controled. If in fact the rate of one of the reactions saturate in excessive nutrient, it becomes a bottleneck reactio. We may assume the first step is extremely slow, i.e $ r_1 \ll r_2 $, then the optimal partition is found to be
\begin{equation}
    \theta_1 \approx \frac{2 r_1}{b}, \quad \theta_2 \approx 1 - \frac{r_1}{2 r_2}, \quad \theta_3 \approx \frac{r_1}{2 r_2}.
\end{equation}
In this case, most of the proteome is allocated to the enzyme of the bottleneck reaction, while little is allocated to the downstream enzymes and the transporter.

\subsection{\label{ssec:arbitrary_mm}Metabolic Network with a Arbitrarily Long Chain}
The general system with $n$ partitions is depicted in figure \ref{fig:n_sector}, and the associated system of ODEs is given in the SI. The analytic solution is best represented as a recursive formula.
\begin{widetext}
    \begin{align}
        Y_1^* &= \frac{1}{2}\left[\sqrt{\left( \alpha_1 + k_1 - 1\right)^2 + 4k_1} - \left(\alpha_1 + k_1 - 1\right) \right], \label{equ:gen_sol_1}\\
        Y_j^* &= \frac{1}{2}\left[\sqrt{\left( \alpha_j + k_1 - S_j \right)^2 + 4k_j} - \left( \alpha_j + k_j - S_j \right) \right], \quad 2 \le j \le n-1, \label{equ:gen_sol_2}\\
        Y_{j}^* &= \theta_{j-n+1} \frac{ \alpha_{n-1} Y_{n-1}^*}{k_{n-1} + Y_{n-1}^*}, \quad n \le j \le 2n-1, \\
        \alpha_j &= \frac{a_j \theta_{j+1}}{b \theta_1}, \quad S_j = 1 - \sum_{r=1}^{j-1} Y_r^*.
        \label{equ:gen_sol_3}
    \end{align}
\end{widetext}

The partition fractions have a ratio according the rate $ \sqrt{k_i / a_i} $ 
\begin{equation}
    \theta_2 \,\colon\, \theta_3 \,\colon\, \cdots \,\colon\, \theta_n = \sqrt{\frac{k_1}{a_1}} \,\colon\, \sqrt{\frac{k_2}{a_2}} \,\colon\, \cdots \,\colon\, \sqrt{\frac{k_{n-1}}{a_{n-1}}}.
\end{equation}
\begin{equation}
    \theta_1 = 1 - A_n\sqrt{b} + \frac{1}{2} A_n^3 b^{3/2} + O(b^{3/2}),
\end{equation}
and the growth rate generalizes nicely to 
\begin{equation}
    \lambda = b - 2A_n b^{3/2} + \left(2A_n^{2}-B_n \right) b^2 + O(b^{5/2}),
\end{equation}
where $A_n = \sum_{j=1}^{n-1}\sqrt{k_j / a_j}$, $ B_n = \sum_{i=1}^{n-1} (1-k_i) / a_i - \sum_{1 \leq i < j \leq n-1} c_i c_j $, and $ c_i = \sqrt{k_i / a_i} $. In the model with $n$ partitions, and hence $n$ metabolic steps leading up to node $x_n$, the dependence of growth rate $\lambda$ on nutrient level $b$ is still linear, the dependence of translational protein fraction $R \equiv Y_{2n-1 *}$ is still square root $b$, and hence the dependence of growth rate $\lambda$ on $R$ is still quadratic. This universal behavior may be interpreted as the fact that $\sqrt{}$ of the nutrient is being allocated to proteome partition section, and the other $\sqrt{}$ to the metabolic pathway itself.

\subsection{Optimality is a Consequence of Balanced Flux and Minimal Waiting Time}
Mathematically, the optimality condition corresponds to the solution that minimizes the waiting time. In the starvation limit, we can write down the leading order waiting time in terms of the model parameters. For the $ i $-th non-terminal node, this is 
\begin{equation}
    \frac{1}{\tau_i} = \theta_{i+1} \left(\frac{a_i Y_i^*}{k_i + Y_i^*}\right) \approx \theta_{i+1} \left(\frac{a_i Y_i^*}{k_i}\right),  
\end{equation}
and thus the total waiting time is $ T = \sum_{i=1}^{n-1} \tau_i $. During starvation, the flux through each node is approximately balanced, so exchanging any two nodes will not change the growth rate and waiting time. We propose the \emph{balanced flux assumption}: the product $ Y_i^* \theta_{i+1} $ is independent of $ Y_j^* \theta_{j+1} $ for all $ j \neq i $. Since each term in the total waiting time is positive, the arithmetic-geometric inequality implies that total waiting time is minimized when
\begin{equation}
    \theta_2 \frac{a_1 Y_1^*}{k_1} = \theta_3 \frac{a_2 Y_2^*}{k_2} = \cdots = \theta_n \frac{a_{n-1} Y_{n-1}^*}{k_{n-1}}.
\end{equation}

This is consistent with our earlier result from SI Section ():
\begin{equation}
    Y_i^* = \left(\frac{k_i \theta_1}{a_i \theta_{i+1}}\right) b, \quad 1 \leq i \leq n-1,
\end{equation}
On the other hand, the results for the nutrient overabundance regime is less straightforward, as the balanced flux assumption no longer holds, and exchanging two nodes will inevitably change the waiting time. 

% Numerical Results
\section{\label{sec:numerical}Numerical Results}

\subsection{Estimate of Parameters for Three-Sector Model}
For realistic predictions and accurate qualitative analysis, we estimate the parameter values $a_1, a_2, k_1, K_2$ using \textit{E. coli} as an example. In a single-cell organism, the number of per-cell protein is approximately $6 \times 10^6$.

Metabolic enzymes make up about $50\%$ of the proteome, so $N_\text{m} \approx 0.5 \times 6\times 10^{6} = 3 \times 10^6$. The average $k_\text{cat}$ is $60 \text{ s}^{-1}$, so $a_1 \approx k_\text{cat}N_\text{m}/N_\text{total} \approx 30 \text{ s}^{-1}.$ The average half-saturation constant for metabolic enzymes is $K_\text{M} \approx 100 \,\mu\text{M}$, $k_2 \approx K_\text{M}N_\text{A}V/N_\text{total} \approx 3.6\times 6\times 10^{-3}$. 

Ribosomes take up about $20\%$ of the proteome, and there are $52$ r-proteins per ribosome, so $N_\text{ribo} \approx 0.2 \times 6 \times 10^6 / 52 \approx 1.7\times 10^4$ ribosomes. Elongation speed is $k_\text{elon} = 16 \text{ a.a./s}$, mean protein length is $L \approx 330 \text{ a.a.}$, volume of an E. coli cell is $V \approx 0.6 \, \mu\text{m}$, so 
\begin{equation}
    \bar{a_2} \approx k_\text{elon} N_\text{ribo} / L N_\text{total} \approx 1.4\times 10^{-4} \text{ s}^{-1}.
\end{equation}
The half-saturation constant for a.a.-tRNA is $K_\text{M} \approx 3 \,\mu \text{m}$, so $k_2 \approx K_\text{M}N_\text{A}V/N_\text{total} \approx 1.8\times 10^{-4}$.

Then $\bar{a}_1 \approx 5$ and $\bar{a}_2 \approx 1.42 \times 10^{-4}$. However, in the step $x_2 \to x_3$, we need about $2100$ metabolites per protein (see SI). Account for stoichiometry and scale by $10^4$ by changing time units, we have $a_1 \approx 23.8, \quad a_2 \approx 1.42$. Environmental nutrient quality is also scaled up, so $b$ is on the magnitude of $1 \sim 10$. For the simulation, we run $b$ over the range $0.01 \sim 100$.

\subsection{Starvation Regime}

\subsection{Nutrient Overabundance Regime}

\subsection{Discussion of Generalized Model}

% Discussion
\section{\label{sec:discussion}Discussion}

\subsection{Interpreting the New Found Relations}

\subsection{Experiments Over the Past Decades}
Furthermore, it has been observed that growth law curves in bacterial physiology generally exhibit two properties: monotonicity and concavity, and the law may be predicted by assuming Michaelis-Menten kinetics and certain global nutrient quality constraints (\cite{Yamagishi2025}). The three-sector model, however, exhibits non-concavity and non-monotonicity for intermediate nutrient qualities. Indeed, it is experimentally observed that growth rate does not always increase with nutrient quality, and there is a nutrient quality threshold past which the growth rate decreases due to a number of biological factors, including toxicity, osmotic stress, and
membrane crowding effects.

%
\begin{acknowledgments}
\end{acknowledgments}

\appendix

\section{Appendixes}


\nocite{*}
\bibliography{draft_sorsamp}% Produces the bibliography via BibTeX.

\end{document}
%
% ****** End of file sorsamp.tex ******
