\documentclass[9pt]{osa-supplemental-document}
\setboolean{shortarticle}{false}

\title{Supplementary Material}
\author{}

\begin{abstract}

\end{abstract}

\setboolean{displaycopyright}{false} % copyright statement should not display in the  supplemental document

\begin{document}

\maketitle
\tableofcontents

\newpage

\section{Introduction}
\subsection{Growth Laws}

\subsection{Scalabe Reaction Networks}
The biomass of a 

In our model, the reaction network can be formulated as the following system of (nonlinear) ODEs of the absolute biomass $ X_i $: 
\begin{equation}
    \begin{split}
        \frac{\mathrm{d}X_1}{\mathrm{d}t} &= bX_3 - \frac{a_1 X_1 X_4}{k_1 N + X_1}, \\
        \frac{\mathrm{d}X_2}{\mathrm{d}t} &= \frac{a_1 X_1 X_4}{k_1 N + X_1} - \frac{a_1 X_2 X_5}{k_2 + X_2}, \\
        \frac{\mathrm{d}X_3}{\mathrm{d}t} &= \theta_1 \frac{a_1 X_2 X_5}{k_2 N + X_2}, \\
        \frac{\mathrm{d}X_4}{\mathrm{d}t} &= \theta_2 \frac{a_1 X_2 X_5}{k_2 N + X_2}, \\
        \frac{\mathrm{d}X_5}{\mathrm{d}t} &= \theta_3 \frac{a_1 X_2 X_5}{k_2 N + X_2},
    \end{split}
\end{equation}
where $ N = X_1 + X_2 + X_3 + X_4 + X_5 $ is the total biomass. In the $ X \to \infty $ limit, $ X_i \sim e^\lambda t $ should exhibit exponential growth, where $ \lambda $ is the growth rate. cell is made up of small molecules (metabolites, amino acids, and peptides) and large molecules (proteins). The total biomass $ N $ is the sum of the absolute biomass of all components in the cell. The \textbf{biomass fraction} of the $ i $-th component is defined as $ Y_i = X_i / N $, so that $ \sum_i Y_i = 1 $. 

Assuming the system has an exponential growth in the long time limit, we have $ N(t) \sim e^{\lambda t} $, where $ \lambda $ is the growth rate. Therefore the \textbf{growth rate} is appropriately defined as 
\begin{equation}
    \lambda = \frac{\mathrm{d} \log N }{\mathrm{d}t} = \frac{1}{N} \frac{\mathrm{d}N}{\mathrm{d}t}.
\end{equation}

In the limit $ t \to \infty $, we should expect the protein fractions $ Y_i $ to approach a steady state, i.e. a fix point of the ODE system. We do a change of variables from biomass space with coordinate $ (X_i, N) $ to the biomass fraction space with coordinate $ Y_i $, a simplex space. Then 
\begin{equation}
    \frac{\mathrm{d} Y_i }{\mathrm{d} t} = \frac{1}{N} \frac{\mathrm{d} X_i}{\mathrm{d} t} - \frac{X_i}{N^2} \frac{\mathrm{d} N}{\mathrm{d} t} = \frac{1}{N} \frac{\mathrm{d} X_i}{\mathrm{d} t} - \lambda Y_i.
\end{equation}

If the biomass ODE system is given by $ \frac{\mathrm{d} \bvec{X}}{\mathrm{d} t} = \bvec{F}(X) $, then the biomass fraction ODE system is conveniently expressed as
\begin{equation}
    \frac{\mathrm{d} \bvec{Y}}{\mathrm{d} t} = \frac{1}{N} \bvec{F}(N \bvec{Y}) - \lambda \bvec{Y}.
\end{equation}

A Scalable Reaction Network (SRN) is defined as a reaction network such that $ \bvec{F}(N \bvec{Y}) = N \bvec{G}(\bvec{Y}) $, i.e. the reaction fluxes scale linearly with the total biomass $ N $. Most biological reaction networks are expected to be SRNs, since it has been observed that 

\subsection{ODE}
The system of ODEs describing for our cell model is 
\begin{subequations}
    \label{equ:Y}
    \begin{align}
        \frac{\mathrm{d}Y_1}{\mathrm{d}t} &= bY_3 - \frac{a_1 Y_1 Y_4}{k_1 + Y_1} - bY_3Y_1, \label{equ:Y1_ODE} \\
        \frac{\mathrm{d}Y_2}{\mathrm{d}t} &= \frac{a_1 Y_1 Y_4}{k_1 + Y_1} - \frac{a_1 Y_2 Y_5}{k_2 + Y_2} - bY_3Y_2, \label{equ:Y2_ODE} \\
        \frac{\mathrm{d}Y_k}{\mathrm{d}t} &= \theta_k \frac{a_1 Y_2 Y_5}{k_2 + Y_2} \; (k=1,2,3). \label{equ:Y345_ODE}
    \end{align}   
\end{subequations}
Here $Y_j$ is the biomass fraction of the $j$-th node in the system, while $Y_3, Y_4, Y_5$ are protein sectors that make up the (three-sector) proteome partition.

\section{Theory}
\subsection{Solution of the Three-Sector Partition Model}

As time tends to infinity, the biomass fractions for this system tend to a steady limit, i.e. no oscillation is observed. So we have the steady-state approximation $ \mathrm{d} Y / \mathrm{d} t \to 0 $ when $ t > T $, and $ Y(t) \approx Y^* $ is the steady-state, which for simplicity we denote by $ Y $ . Then equations (\ref{equ:Y}) become 

% Main ODE equations 
\begin{subequations}
    \label{equ:Y_steady}
    \begin{align}
        b Y_3 - \frac{a_1 Y_1 Y_4}{ k_1 + Y_1} - b Y_3 Y_1 &= 0, \label{equ:Y1_steady} \\
        \frac{a_1 Y_1 Y_4}{ k_1 + Y_1} - \frac{a_2 Y_2 Y_5}{ k_2 + Y^*_2} - b Y_3 Y_2 &= 0, \label{equ:Y2_steady} \\
        \theta_1 \left(\frac{ a_2 Y_2 Y_5}{ k_1 + Y_2}\right) - b (Y_3)^2 &= 0, \label{equ:Y3_steady} \\
        \theta_2 \left(\frac{ a_2 Y_2 Y_5}{ k_1 + Y_2}\right) - b Y_3 Y_4 &= 0, \label{equ:Y4_steady} \\
        \theta_3 \left(\frac{ a_2 Y_2 Y_5}{ k_1 + Y_2}\right) - b Y_3 Y_5 &= 0. \label{equ:Y5_steady} \\
    \end{align}
\end{subequations}
Since $ Y_3, Y_4, Y_5 \neq 0 $, the equations (\ref{equ:Y3_steady}), (\ref{equ:Y4_steady}), and (\ref{equ:Y5_steady}) show that 
\[
    Y_3 \colon Y_4 \colon Y_5 = \theta_1 \colon \theta_2 \colon \theta_3.
\]
Work with $ Y_3 $, so $ Y_4 = (\theta_2 / \theta_1) Y_3 $ and $ Y_5 = (\theta_3 / \theta_1) Y_3 $. Then the first two equations become 
\begin{subequations}
    \begin{align}
        (Y_1)^{2} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 - 1 \right) Y_1 - k_1 &= 0, \\
        (Y_2)^{2} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 - (1 - Y_1) \right) Y_2 - (1 - Y_1 ) k_{2} &= 0. \\ 
    \end{align}
\end{subequations} 

Equation (\ref{equ:Y5_steady}) gives
\begin{equation}
    Y_3 = \left(\frac{a_2 \theta_3}{b}\right) \left(\frac{Y_2}{k_2 + Y_2}\right).
\end{equation}
That is, given $ ( \theta_1 , \theta_2 , \theta_3 ) $, we have an analytic solution of the steady state $ Y = Y[\theta_1, \theta_2, \theta_3] $, given by equation (\ref{equ:Y_steady_sol}): 

\begin{equation*}
    1 - Y_1 = \frac{1}{2} \left[ -\sqrt{\left(\frac{a_1 \theta_2}{b \theta_1}+ k_1 - 1\right)^{2} + 4k_1} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 + 1\right) \right]. 
\end{equation*}

% \notag disables numbering
% This is the analytic solution of the system
\begin{subequations}
    \label{equ:Y_steady_sol}
    \begin{align}
        %
        Y_1 &= \frac{1}{2} \left[ \sqrt{\left(\frac{a_1 \theta_2}{b \theta_1} + k_1 - 1\right)^{2} + 4k_1} - \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 - 1\right) \right], \\
        %
        Y_2 &= \frac{1}{2} \left[ \sqrt{\left(\frac{a_2 \theta_3}{b \theta_1} + k_2 - (1 - Y_1 )\right)^{2} + 4 (1 - Y_1) k_2} - \left(\frac{a_1 \theta_2}{b \theta_1} + k_2 - (1 - Y_1 ) \right) \right] \notag \\ 
        &= \frac{1}{2} \sqrt{
            \begin{aligned}
                & \left\{ \frac{a_2 \theta_3}{b \theta_1} + k_2 - \frac{1}{2} \left[ -\sqrt{\left(\frac{a_1 \theta_2}{b \theta_1}+ k_1 - 1\right)^{2} + 4k_1} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 + 1\right) \right] \right\}^{2} + \\
                &4k_1 \left\{ \frac{1}{2} \left[ -\sqrt{\left(\frac{a_1 \theta_2}{b \theta_1}+ k_1 - 1\right)^{2} + 4k_1} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 + 1\right) \right] \right\} \\
            \end{aligned}
        } \notag \\ 
        & - \frac{1}{2} \left\{ \frac{a_2 \theta_3}{b \theta_1} + k_2 - \frac{1}{2} \left[ - \sqrt{ \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 -1\right)^{2} + 4 k_1} + \left(\frac{a_1 \theta_2}{b \theta_1} + k_1 + 1\right) \right] \right\}, \\
        %
        Y_3 &= \left( \frac{a_2 \theta_3}{b}\right) \left(\frac{Y_2}{k_2 + Y_2}\right), \\
        %
        Y_4 &= \left( \frac{a_2 \theta_2 \theta_3}{b \theta_1}\right) \left(\frac{Y_2}{k_2 + Y_2}\right), \\
        %
        Y_5 &= \left( \frac{a_2 \theta_3^{2}}{b \theta_1}\right) \left(\frac{Y_2}{k_2 + Y_2}\right).
    \end{align}
\end{subequations}
This is an expression of $Y = Y(\theta_1, \theta_2, \theta_3)$, where $\theta$ is undetermined. The main result expresses $\theta$ in terms of system parameters $a_1, a_2, b, k_1, k_2$, and hence $Y$ in terms of $a_1, a_2, b, k_1, k_2$. In the above solution, the minus sign solution of the quadratic equations 
\begin{align}
    Y_1^2 + \left(\frac{a_1\theta_2}{b\theta_1} + k_1 - 1\right)Y_1 - k_1 &= 0 , \\
    Y_2^2 + \left[\frac{a_2\theta_3}{b\theta_1} + k_2 - \left(1 - Y_1\right)\right]Y_2 - k_2\left(1 - Y_1\right) &= 0
\end{align}
is discarded, since biomass fraction must be positive.

\subsection{Derivation of the Partition Strength in the Low Nutrient Limit}
The partition strength is given here up to order $O(b^2)$:
\begin{align}
    \theta_1 &= 1 - A\sqrt{b} - \frac{1}{2}A\left(2C - 3A^2 + 4D\right)b^{3/2} + O(b^2),\\
    \theta_2 &= \sqrt{\frac{k_1}{a_1}} \left[\sqrt{b} - \frac{1}{2}A^2 b^{3/2} - \left(AC - A^3 + 2AD\right)b^2 \right] + O(b^{5/2}), \\
    \theta_3 &= \sqrt{\frac{k_2}{a_2}} \left[\sqrt{b} - \frac{1}{2}A^2 b^{3/2} - \left(AC - A^3 + 2AD\right)b^2 \right] + O(b^{5/2}),
\end{align}

where 
\begin{equation}
    A = \sqrt{\frac{k_1}{a_1}} + \sqrt{\frac{k_2}{a_2}}, \quad 
    B = \frac{k_1}{a_1} + \frac{k_2}{a_2}, \quad 
    C = \frac{a_1}{k_1} + \frac{a_2}{k_2}, \quad 
    D = \sqrt{\frac{k_1k_2}{a_1 a_2}}, \quad 
    E = \frac{1}{a_1}+\frac{1}{a_2}.
\end{equation}
are symmetric in $(k_1, a_1), (k_2, a_2)$. To derive this, consider the series expansion of $Y_1, Y_2, Y_3$ in the limit $b \to 0$. 

\begin{align}
    Y_1 &= \left(\frac{k_1\theta_1}{a_1\theta_2}\right)b + \left[\frac{k_1 \theta_1^2 (1-k_1)}{a_1^2 \theta_2^2}\right]b^2 + O(b^3), \\
    Y_2 &= \left(\frac{k_2\theta_1}{a_2\theta_3}\right)b - \left[\frac{k_2\theta_1^2\left(k_1a_2\theta_3 + k_2a_1\theta_2 - a_1\theta_2\right)}{a_1 a_2^2 \theta_2 \theta_3^2}\right]b^2 + O(b^3), \\
    Y_3 &= \theta_1 - b\theta_1^2\left(\frac{k_1}{a_1\theta_2} + \frac{k_2}{a_2\theta_3}\right) + b^2\theta_1^3 \left[\frac{k_1 k_2}{a_1 a_2\theta_2\theta_3} - \frac{k_1(1-k_1)}{a_1^2\theta_2^2} - \frac{k_2(1-k_2)}{a_2^2\theta_3^2}\right].
\end{align}

Therefore 
\begin{equation}
    \lambda = b\theta_1 - b^2\theta_1^2\left(\frac{k_1}{a_1\theta_2} + \frac{k_2}{a_2\theta_3}\right) + b^3\theta_1^3 \left[\frac{k_1 k_2}{a_1 a_2\theta_2\theta_3} - \frac{k_1(1-k_1)}{a_1^2\theta_2^2} - \frac{k_2(1-k_2)}{a_2^2\theta_3^2}\right].
\end{equation}

By the method of Lagrange multipliers, set the Lagrangian to $L(\theta) = \lambda(\theta) - \mu \left(\theta_1 + \theta_2 + \theta_3 - 1\right)$ with Lagrange multiplier $\mu$. Then 
\begin{align}
    \frac{\partial L}{\partial \theta_1} &= b - 2b^2\theta_1 \left(\frac{k_1}{a_1\theta_2} + \frac{k_2}{a_2\theta_3}\right) + 3b^3 \theta_1^2\left[\frac{k_1 k_2}{a_1 a_2\theta_2\theta_3} - \frac{k_1(1-k_1)}{a_1^2\theta_2^2} - \frac{k_2(1-k_2)}{a_2^2\theta_3^2}\right] - \mu, \label{equ:lagrange1}\\
    \frac{\partial L}{\partial \theta_2} &= \left(\frac{k_1\theta_1^2}{a_1\theta_2^2}\right) b^2 - \theta_1^3\left[\frac{k_1k_2}{a_1a_2\theta_2\theta_3} - \frac{2k_1(1-k_1)}{a_1^2\theta_2^3}\right]b^3 - \mu, \label{equ:lagrange2} \\
    \frac{\partial L}{\partial \theta_3} &= \left(\frac{k_2\theta_1^2}{a_2\theta_3^2}\right) b^2 - \theta_1^3\left[\frac{k_1k_2}{a_1a_2\theta_2\theta_3} - \frac{2k_2(1-k_2)}{a_2^2\theta_3^3}\right]b^3 - \mu. \label{equ:lagrange3} \\
\end{align}
Keep only terms of order up to $O(b^2)$ and solve equations (\ref{equ:lagrange2}, \ref{equ:lagrange3}), we get an important relationship that generalizes to the $n$-sector model,
\begin{equation}
    \frac{\theta_2}{\theta_3} = \sqrt{\frac{k_1 a_2}{k_2 a_1}}.
\end{equation}
Then equation (\ref{equ:lagrange1}) becomes a quadratic in $\theta_1/\theta_2$:
\begin{equation}
    \left(\frac{bk_1}{a_1}\right)\left(\frac{\theta_1}{\theta_2}\right)^2 + 2b\sqrt{\frac{k_1}{a_1}}A\left(\frac{\theta_1}{\theta_2}\right) - 1 = 0,
\end{equation}
where $A = \sqrt{\frac{k_1}{a_1}} + \sqrt{\frac{k_2}{a_2}}$. The solution admits an asymptotic expansion in the form of a Puiseux series:
\begin{equation}
    r \equiv \frac{\theta_1}{\theta_2} = \sqrt{\frac{a_1}{k_1}}\left(\frac{1}{\sqrt{b}} - A + \frac{1}{2}A^2\sqrt{b}\right) + O(b),
\end{equation}
and
\begin{equation}
    s \equiv \frac{\theta_1}{\theta_3} = \sqrt{\frac{a_2}{k_3}}\left(\frac{1}{\sqrt{b}} - A + \frac{1}{2}A^2\sqrt{b}\right) + O(b).
\end{equation}
The symmetry in the equations hints that we should solve for $\theta_1$ first, then find $\theta_2 = \theta_1/r, \theta_3 = \theta_1/s$. We expand up to order $O(b^{3/2})$ and solve for $ \theta_1 $. 
\begin{equation}
    \theta_1 = (1 + 1/r + 1/s)^{-1},
\end{equation} 
then 
\begin{subequations}
    \begin{align}
        \theta_1 &= 1 - A\sqrt{b} - \frac{1}{2} A^3 b^{3/2} + O(b^2), \\ 
        \theta_2 &= \sqrt{\frac{k_1}{a_1}}\left(\sqrt{b} - \frac{1}{2}A^{2} b^{3/2} \right) + O(b^{2}), \\
        \theta_3 &= \sqrt{\frac{k_2}{a_2}}\left(\sqrt{b} - \frac{1}{2}A^{2} b^{3/2} \right) + O(b^{2}). 
    \end{align}
\end{subequations}

\subsection{Derivation of Protein Fraction and Growth Rate During Starvation}
\subsubsection{Trajectories and Growth Rate}
The growth rate $\lambda$ is given as a perturbative series in $\sqrt{b}$:
\begin{equation}
    \lambda = b - 2Ab^{3/2} + \left[2A^2 + B + D -E\right]b^2, \label{equ:growth_rate}
\end{equation}
which is easily derived by plugging in equations for $\theta$. This formula makes intuitively sense, since it is invariant under exchange of indices (i.e. $(a_1, k_1) \to (a_2, k_2)$), as will be observed for many quantities below.

Substitute in expressions for $\theta$ to get
\begin{align}
    Y_1 &= \sqrt{\frac{k_1}{a_1}}\sqrt{b} + \left(\frac{1}{a_1}(1-k_1) - A\sqrt{\frac{k_1}{a_1}}\right)b + \sqrt{\frac{k_1}{a_1}}A^2 b^{3/2} + O(b^2), \\
    Y_2 &= \sqrt{\frac{k_2}{a_2}}\sqrt{b} + \left(\sqrt{\frac{k_2}{a_2}}A + D - \frac{(1-k_2)}{a_2}\right)b + \left[\frac{1}{2}\sqrt{\frac{k_2}{a_2}}A^2 + 2AD - 2A\frac{(1-k_2)}{a_2}\right]b^{3/2} + O(b^2), \\
    Y_3 &= 1 - 2A\sqrt{b} + \left[2A^2 + B + D -E\right]b + O(b^{3/2}), \\
    Y_4 &= \sqrt{\frac{k_1}{a_1}}\left[\sqrt{b} - Ab + \left(\frac{1}{2}A^2 + B + D - E\right)b^{3/2}\right] + O(b^2), \\
    Y_5 &= \sqrt{\frac{k_2}{a_2}}\left[\sqrt{b} - Ab + \left(\frac{1}{2}A^2 + B + D - E\right)b^{3/2}\right] + O(b^2). \label{equ:Y5}
\end{align}

We also have the fraction of small molecules (intermediate metabolites, amino acids, and peptides) and of large molecules (transport/metabolic/ribosomal and translational proteins), denoted $\text{Small}$ and $\text{Large}$ respectively.
\begin{align}
    \text{Small} &= A\sqrt{b} - \left(A^2+B+D-E\right)b + O(b^{3/2}), \\
    \text{Large} &= 1 - \text{Small} = 1 - A\sqrt{b} + \left(A^2+B+D-E\right)b + O(b^{3/2}).
\end{align}
These are also invariant under the exchange of indices, so the rate of nutrient processing does not affect the overall fraction of products.

\subsubsection{Growth Rate and Ribosomal Protein Fraction}
Equations (\ref{equ:growth_rate}) and (\ref{equ:Y5}) express $\lambda$ and $Y_5$ in terms of nutrient level $b$. Recall the theory of \textbf{series reversion}: Suppose \( y = f(x) = a_1 x + a_2 x^2 + a_3 x^3 + a_4 x^4 + a_5 x^5 + \cdots \). Then the inverse series \( x = f^{-1}(y) \) up to the $5$th order is:
\begin{equation*}
    \begin{aligned}
    x &= \frac{1}{a_1} y 
        - \frac{a_2}{a_1^3} y^2 
        + \frac{2a_2^2 - a_1 a_3}{a_1^5} y^3 \\
      &\quad + \frac{5a_1 a_2 a_3 - 5a_2^3 - a_1^2 a_4}{a_1^7} y^4 \\
      &\quad + \frac{-6a_2^4 + 9a_1 a_2^2 a_3 - 3a_1^2 a_3^2 - 4a_1^2 a_2 a_4 + a_1^3 a_5}{a_1^9} y^5 + \cdots
    \end{aligned}
\end{equation*}

We have
\begin{equation}
    \sqrt{b} = \sqrt{\frac{a_2}{k_2}}R + A\left(\sqrt{\frac{a_2}{k_2}}R\right)^2 + \left(\frac{3A^2 - 2B - 2D + 2E}{2}\right)\left(\sqrt{\frac{a_2}{k_2}}R\right)^3 + O(R^4).
\end{equation}

Then the growth rate can be written in terms of ribosomal protein fraction $R$ if we plug in $\sqrt{b}$:
\begin{equation}
    \lambda = \frac{a_2}{k_2}R^2 - \left(\frac{a_2}{k_2}\right)^2 (B+D-E)R^4 + \left(\frac{a_2}{k_2}\right)^{5/2}8A(B+D-E)R^5 + O(R^6).
\end{equation}

\subsection{Discussion of the General System}
\subsubsection{Solution}
The $n$-sector partition model has $2n-1$ nodes and fluxes, and obeys the following system of $2n-1$ ODEs:
\begin{subequations}
    \begin{align}
        \frac{\mathrm{d} Y_1}{\mathrm{d} t} &= b Y_n - \frac{a_1 Y_1 Y_{n+1}}{k_1 + Y_1} - bY_n Y_1, \\ 
        \frac{\mathrm{d}Y_2}{\mathrm{d}t} &= \frac{a_1 Y_1 Y_{n+1}}{k_1 + Y_1} - \frac{a_1 Y_2 Y_{n+2}}{k_2 + Y_2} - bY_n Y_2 , \\ 
        &\vdots \notag \\
        \frac{\mathrm{d}Y_{n-1}}{\mathrm{d}t} &= \frac{a_{n-2} Y_{n-2} Y_{2n-2}}{k_{n-2} + Y_{n-2}} - \frac{a_{n-1} Y_{n-1} Y_{2n-1}}{k_{n-1} + Y_{n-1}} - b Y_n Y_{n-1}, \\
        \frac{\mathrm{d}Y_n}{\mathrm{d}t} &= \theta_1 \left(\frac{a_{n-1} Y_{n-1} Y_{2n-1}}{k_{n-1} + Y_{n-1}}\right) - b Y_n^2 = 0, \\
        & \vdots \notag \\
        \frac{\mathrm{d}Y_{2n-1} }{\mathrm{d}t} &= \theta_n \left(\frac{a_{n-1} Y_{n-1} Y_{2n-1}}{k_{n-1} + Y_{n-1}}\right) - b Y_n Y_{2n-1} = 0. 
    \end{align}
\end{subequations}

Following the same algebraic method as before, the solution is given by equations (\ref{equ:gen_sol_1}), (\ref{equ:gen_sol_2}), and (\ref{equ:gen_sol_3}).
\begin{subequations}
    \label{equs:n_sector_sol}
    \begin{align}
        Y_1 &= \frac{1}{2}\left[\sqrt{\left(\frac{a_1\theta_2}{b\theta_1} + k_1 - 1\right)^2 + 4k_1} - \left(\frac{a_1\theta_2}{b\theta_1} + k_1 - 1\right) \right], \label{equ:gen_sol_1}\\
        Y_j &= \frac{1}{2}\left[\sqrt{\left(\frac{a_j\theta_{j+1}}{b\theta_1} + k_1 - \left(1-\sum^{j-1}_{r=1}Y_r\right)\right)^2 + 4k_j} - \left(\frac{a_j\theta_{j+1}}{b\theta_1} + k_j - \left(1-\sum^{j-1}_{r=1}Y_r\right)\right) \right] \; (2 \le j \le n-1), \label{equ:gen_sol_2}\\
        Y_{j} &= \theta_{j-n+1} \left(\frac{a_{n-1}\theta_{n}}{b\theta_1}\right)\frac{Y_{n-1}}{k_{n-1} + Y_{n-1}} \; (n \le j \le 2n-1). \label{equ:gen_sol_3} \\
    \end{align}
\end{subequations}

Looking at the solution, it is true that 
\begin{equation}
    \sqrt{\left(\frac{a_j\theta_{j+1}}{b\theta_1} + k_1 - \left(1-\sum^{j-1}_{r=1}Y_r\right)\right)^2 + 4k_j} \ge \left| \frac{a_j\theta_{j+1}}{b\theta_1} + k_1 - \left(1-\sum^{j-1}_{r=1}Y_r\right)\right|,
\end{equation}
so the solution with $-$ sign is always negative and hence not physical. By the same reasoning, all $Y_j$ for $1 \le j \le n-1$ are positive, hence all $Y_j$ for $1 \le j \le 2n-1$ are positive. This makes sure there is a unique global steady state, and such a state always exists.

\subsubsection{Example Calculation for $n=4$}
The solution to the general case has been discussed above, so here we elaborate on details of the generalization with the case $n=4$. For sake of brevity, higher partitions will be omitted, but the techniques and notation introduced in this example can easily generalize by means of mathematical induction.

Using the general solution for $n$-sector partition equations (\ref{equs:n_sector_sol}), the solution for four-sector partition is
\begin{subequations}
    \begin{align}
        Y_1 &= \frac{1}{2}\left[\sqrt{\left(\frac{a_1\theta_2}{b\theta_1} + k_1 - 1\right)^2 + 4k_1} - \left(\frac{a_1\theta_2}{b\theta_1} + k_1 - 1\right)\right], \\
        Y_2 &= \frac{1}{2}\left\{\sqrt{\left[\frac{a_2\theta_3}{b\theta_1} + k_2 - \left(1-Y_1\right)\right]^2 + 4k_2\left(1 - Y_1\right)} - \left[\frac{a_2\theta_3}{b\theta_1} + k_2 - \left(1 - Y_1\right)\right]\right\}, \\
        Y_3 &= \frac{1}{2}\left\{\sqrt{\left[\frac{a_3\theta_4}{b\theta_1} + k_3 - \left(1-Y_1-Y_2\right)\right]^2 + 4k_3\left(1 - Y_1 - Y_2\right)} - \left[\frac{a_3\theta_4}{b\theta_1} + k_3 - \left(1 - Y_1 - Y_2\right)\right]\right\}, \\
        Y_4 &= \left(\frac{a_3 \theta_4}{b}\right) \frac{Y_3}{k_3 + Y_3}, \\
        Y_5 &= \frac{\theta_2}{\theta_1}Y_4, \quad Y_6 = \frac{\theta_3}{\theta_1}Y_4, \quad Y_7 = \frac{\theta_4}{\theta_1}Y_4.
    \end{align}
\end{subequations}

Expand in the limit of small $b$, then the transport and metabolic molecules have fractions 
\begin{subequations}
    \begin{align}
        Y_1 &= \frac{bk_1\theta_1}{a_1 \theta_2}\left[1 + (1-k_1)\frac{b\theta_1}{a_1\theta_2}\right] + O(b^3), \\
        Y_2 &= \frac{bk_2\theta_1}{a_2\theta_3}\left[1 + b\theta_1\left(\frac{1-k_2}{a_2\theta_3} - \frac{k_1}{a_1\theta_2}\right)\right] + O(b^3), \\
        Y_3 &= \theta_1 \left[1 - b\theta_1 \left(\frac{k_1}{a_1\theta_2} + \frac{k_2}{a_2\theta_3} + \frac{k_3}{a_3\theta_4}\right)\right] + O(b^2).
    \end{align}
\end{subequations}

The form of the growth rate 
\begin{equation}
    \lambda = b\theta_1 \left[1 - b\theta_1 \left(\frac{k_1}{a_1\theta_2} + \frac{k_2}{a_2\theta_3} + \frac{k_3}{a_3\theta_4}\right)\right] + O(b^3)
\end{equation}
is similar to that of the three-sector case. Let $A_4 = \sqrt{\frac{k_1}{a_1}} + \sqrt{\frac{k_2}{a_2}} + \sqrt{\frac{k_3}{a_3}}$. Carrying out the same computations of Lagrange multipliers, we find 
\begin{equation}
    \theta_2 \;\colon\; \theta_3 \;\colon\; \theta_4 = \sqrt{\frac{k_1}{a_1}} \;\colon\; \sqrt{\frac{k_2}{a_2}} \;\colon\; \sqrt{\frac{k_3}{a_3}},
\end{equation}
and
\begin{subequations}
    \begin{align}
        \frac{\theta_1}{\theta_2} &= \sqrt{\frac{a_1}{k_1}}\left(\frac{1}{\sqrt{b}} - A_4 + \frac{1}{2}A_4^2\sqrt{b}\right) + O(b), \\
        \frac{\theta_1}{\theta_3} &= \sqrt{\frac{a_2}{k_2}}\left(\frac{1}{\sqrt{b}} - A_4 + \frac{1}{2}A_4^2\sqrt{b}\right) + O(b), \\
        \frac{\theta_1}{\theta_3} &= \sqrt{\frac{a_3}{k_3}}\left(\frac{1}{\sqrt{b}} - A_4 + \frac{1}{2}A_4^2\sqrt{b}\right) + O(b).
    \end{align}
\end{subequations}
Solve for $\theta_1$ by normalization, then solve for $\theta_{j=2, 3, 4}$. Generalize the index-invariant variables

Then
\begin{subequations}
    \begin{align}
        \theta_1 &= \left(1 + \frac{\theta_2}{\theta_1} + \frac{\theta_3}{\theta_1} + \frac{\theta_4}{\theta_1}\right)^{-1} = 1 - A_4\sqrt{b} - \frac{1}{2}A_4\left(2C_4 - 3A_4^2 + 4D_4\right)b^{3/2} + O(b^2), \\
        \theta_2 &= \sqrt{\frac{k_1}{a_1}} \left[\sqrt{b} - \frac{1}{2}A_4^2 b^{3/2} - \left(A_4C_4 - A_4^3 + 2A_4D_4\right)b^2 \right] + O(b^{5/2}), \\
        \theta_3 &= \sqrt{\frac{k_2}{a_2}} \left[\sqrt{b} - \frac{1}{2}A_4^2 b^{3/2} - \left(A_4C_4 - A_4^3 + 2A_4D_4\right)b^2 \right] + O(b^{5/2}), \\
        \theta_4 &= \sqrt{\frac{k_3}{a_3}} \left[\sqrt{b} - \frac{1}{2}A_4^2 b^{3/2} - \left(A_4C_4 - A_4^3 + 2A_4D_4\right)b^2 \right] + O(b^{5/2}).
    \end{align}
\end{subequations}
With the new expressions for $\theta$, substitution followed by expansion up to leading orders gives similar formulae as in the three-sector case for other physical quantities, but with the variables $A, B, C, D, E$ replaced by their respective generalizations $A_4, B_4, C_4, D_4, E_4$ and the addition of a fourth component $\theta_4$.

In particular, we have 
\begin{equation}
    \lambda = b - 2A_4 b^{3/2} + \left(2A_4^2 + B_4 + D_4 - E_4\right)b^2 + O(b^{5/2}),
\end{equation}
which is invariant under permutation of indices $1 \leftrightarrow 2 \leftrightarrow 3$.

\subsection{Derivation of Protein Fraction and Growth Rate in High Nutrient Environment}
\subsubsection{Trajectories and Growth Rate}
Large supply or demand limits yield corner solutions. In optimal resource allocation (e.g. flux-balance analysis), an unconstrained input leads to one pathway dominating flux. in a high-$b$ regime the metabolic sector $Y_4$ can hardly operate, while the ribosomal/translational sector dominates. Biologically, this implies nearly all proteome is allocated for ribosomes and translational molecules, shutting down metabolic enzyme production. 

The partition strength is given here up to order $O(b^{-3})$:
\begin{subequations}
    \begin{align}
        \theta_1 &= \frac{1}{(2\alpha + \beta\alpha^2)b} + \frac{1}{(2\alpha + \beta\alpha^2)^2b^2} + O(b^{-3}),\\
        \theta_2 &= \frac{\alpha}{2\alpha + \beta\alpha^2} - \frac{\alpha}{(2\alpha + \beta\alpha^2)^2b} + \frac{\alpha}{(2\alpha + \beta\alpha^2)^3 b^2} + O(b^{-3}),\\
        \theta_3 &= \frac{\alpha + \beta\alpha^2}{2\alpha + \beta\alpha^2} - \frac{\alpha + \beta\alpha^2}{(2\alpha + \beta\alpha^2)^2b} + \frac{\alpha + \beta \alpha^2}{(2\alpha + \beta\alpha^2)^3 b^2} + O(b^{-3}),\\
        \alpha &= \frac{k_2^2(1+k_1)^4a_2}{(1+k_1+k_1k_2)^2a_1^2 - k_2(1+k_1)^2a_1a_2 + (1+k_1)^4(1+k_2)a_2^2}, \\
        \beta &= \frac{(1+k_1+k_1k_2)a_1 + (1+k_1)^2a_2}{(1+k_1)^2k_2}.
    \end{align}
\end{subequations}

Note that $\theta_1 + \theta_2 + \theta_3 = 1$ up to order $O(b^{-2}$. To derive this, consider the asymptotic expansion of $Y_1, Y_2, Y_3$ in the limit $b \to \infty$.

\begin{subequations}
    \begin{align}
        Y_1 &= 1 - \frac{1}{1+k_1}\left(\frac{a_1\theta_2}{b\theta_1}\right) + \frac{k_1}{(1+k_1)^3}\left(\frac{a_1\theta_2}{b\theta_1}\right)^2, \\
        Y_2 &= \frac{1}{1+k_1}\left(\frac{a_1\theta_2}{b\theta_1}\right) - \frac{1}{(1+k_1)^3}\left[k_1 + \frac{(1+k_1)^2a_2\theta_3}{k_2a_1\theta_2}\right]\left(\frac{a_1\theta_2}{b\theta_1}\right)^2, \\
        Y_3 &= \frac{a_1 a_2}{(1+k_1)k_2 b^2}\left(\frac{\theta_2\theta_3}{\theta_1}\right) - \frac{a_1 a_2}{(1+k_1)^3 k_2^2}\left(\frac{\theta_2\theta_3}{\theta_1^2}\right) \left[(1+k_1)^2a_2\theta_3 + (1+k_1+k_1k_2)a_1\theta_2\right]\frac{1}{b^3}.
    \end{align}
\end{subequations}

The growth rate is
\begin{equation}
    \lambda = \frac{a_1 a_2}{(1+k_1)k_2 b}\left(\frac{\theta_2\theta_3}{\theta_1}\right) - \frac{a_1 a_2}{(1+k_1)^3k_2^2b^2}\left(\frac{\theta_2\theta_3}{\theta_1^2}\right) \left[(1+k_1)^2a_2\theta_3 + (1+k_1+k_1k_2)a_1\theta_2\right].
\end{equation}

Again by the method of Lagrange multipliers, set the Lagrangian to $L(\theta) = \lambda(\theta) - \mu(\theta_1 + \theta_2 + \theta_3 - 1)$ with Lagrange multiplier $\mu$. Then 
\begin{subequations}
    \begin{align}
        \frac{\partial L}{\partial \theta_1} &= -\frac{a_1a_2\theta_2\theta_3}{(1+k_1)k_2\theta_1^2b} + \frac{2a_1a_2\theta_2\theta_3\left[(1+k_1+k_1k_2)a_1\theta_2 + (1+k_1)^2a_2\theta_3\right]}{(1+k_1)^3k_2^2\theta_1^3b^2} - \mu, \label{equ:lagrange_high1}\\ 
        \frac{\partial L}{\partial\theta_2} &= \frac{a_1a_2\theta_3}{(1+k_1)k_2\theta_1 b} - \frac{a_1a_2\theta_3\left[2(1+k_1+k_1k_2)a_1\theta_2 + (1+k_1)^2a_2\theta_3\right]}{(1+k_1)^3k_2^2\theta_1^2b^2} - \mu, \label{equ:lagrange_high2}\\
        \frac{\partial L}{\partial\theta_3} &= \frac{a_1a_2\theta_3}{(1+k_1)k_2\theta_1 b} - \frac{a_1a_2\theta_2\left[(1+k_1+k_1k_2)a_1\theta_2 + 2(1+k_1)^2a_2\theta_3\right]}{(1+k_1)^3k_2^2\theta_1^2b^2} - \mu. \label{equ:lagrange_high3}
    \end{align}
\end{subequations}

Let $p = \theta_3/\theta_1$, $q = \theta_2/\theta_1$, $P \equiv r/b, Q = q/b$. From equations (\ref{equ:lagrange_high2}) and (\ref{equ:lagrange_high3}), up to order $O(b^{-2})$ we have 
\begin{align}
    &\frac{a_1 a_2^2}{(1+k_1)k_2^2}P^2 + \left[\frac{a_1a_2}{(1+k_1)k_2} - \frac{2a_1a_2\left((1+k_1+k_1k_2)a_1 + (1+k_1)^2a_2\right)}{(1+k_1)^3k_2^2}Q\right]P \\
    &\quad + \left[\frac{a_1^2a_2(1+k_1+k_1k_2)}{(1+k_1)^3k_2^2}Q^2 - \frac{a_1a_2}{(1+k_1)k_2}Q\right] = 0.
\end{align}
Solve for $P$ and keep terms up to $O(b^{-2})$, i.e. $Q^2$.
\begin{align}
    P &= Q + \frac{(1+k_1+k_1k_2)a_1 + (1+k_1)^2a_2}{(1+k_1)^2k_2}Q^2 + O(Q^3) \notag \\
    &\equiv Q + \beta Q^2 + O(Q^3).\label{equ:lagrange_high4}
\end{align}
Notice that when $Q$ is small, this reduces to the previous (unphysical) result. However, the coefficient of $Q^2$ is approximately $\sim 40000$ for $a_1 = 23.8, k_1 = 0.1, a_2 = 1.42, k_2 = 0.003$, while $Q \sim 0.01/b$, so the expression is reasonable. Plug equation (\ref{equ:lagrange_high4}) into equation (\ref{equ:lagrange_high1}) and use equation (\ref{equ:lagrange_high2}), we have
\begin{align}
    &\left[\frac{a_1a_2}{(1+k_1)k_2} - \frac{2a_1^2a_2(1+k_1+k_1k_2)}{(1+k_1+k_1k_2)^3k_2^2}  - \frac{a_1a_2^2}{(1+k_1)k_2} \right] \left[Q + \frac{(1+k_1+k_1k_2)a_1 + (1+k_1)^2 a_2}{(1+k_1)^2k_2}Q^2\right] \\
    &= \frac{a_1a_2}{(1+k_1)k_2}Q - \frac{a_1^2 a_2(1+k_1+k_1k_2)}{(1+k_1)^3k_2}Q^2 - \frac{2a_1a_2^2}{(1+k_1)k_2}Q\left[Q + \frac{(1+k_1+k_1k_2)a_1 + (1+k_1)^2a_2}{(1+k_1)^2k_2}Q^2\right].
\end{align}
Leading factor $O(Q)$ cancels out, and dividing by $Q$ gives
\begin{equation}
    Q = \frac{k_2^2(1+k_1)^4a_2}{(1+k_1+k_1k_2)^2a_1^2 - k_2(1+k_1)^2a_1a_2 + (1+k_1)^4(1+k_2)a_2^2} \equiv \alpha. \label{equ:Q}
\end{equation}
By equation (\ref{equ:lagrange_high4}), 
\begin{equation}
    P = \alpha + \beta\alpha^2 = O(1), \label{equ:P}
\end{equation}
so both $\theta_2/\theta_1$ and $\theta_3/\theta_1$ are $O(b)$. To determine the specific dependencies, solve for $\theta_1$ with
\begin{equation}
    \theta_1 = \left(1 + \frac{\theta_2}{\theta_1} + \frac{\theta_3}{\theta_1}\right)^{-1} = \frac{1}{(2\alpha + \beta\alpha^2)b} - \frac{1}{(2\alpha + \beta\alpha^2)^2b^2} + O(b^{-3}).
\end{equation}
up to leading order. Then $\theta_2$ and $\theta_3$ can be derived from equations (\ref{equ:P}) and (\ref{equ:Q}).


\subsection{Derivation of Partition Strengths in the Bottleneck Limit}
In a metabolic chain of $ m $ intermediate reactions, let $ m $ denote the single \textbf{bottleneck reaction} , while all other $ a_i $'s are of similar magnitude. Consider as a simple example the case of $n=3$ and $m=1$, with the transport ($ b $ ) and last reaction ($ a_2 $ ) being fast and the middle reaction ($ a_1 $ ) being slow. 

We begin with the three-sector model, where their analytic solution has been worked out. 

\subsubsection{First Metabolic Step Limited}
If the first metabolic step is a bottleneck reaction, following the same procedure as before, we can expand in the limit of small $ a_1 $. 

The third-order expansions are 
\begin{equation}
    \begin{split}
        %
        Y_1 &= 1 + \left(-\frac{\theta _{2}}{b\,\theta _{1}\,\left(k_{1}+1\right)}\right)\,a_{1} + \frac{k_{1}\,{\theta _{2}}^2}{b^2\,{\theta _{1}}^2\,{\left(k_{1}+1\right)}^3}\,{a_{1}}^2, \\
        %
        Y_2 &= \frac{k_{2}\,\theta _{2}}{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)\,\left(k_{1}+1\right)}\,a_{1} \\
        &+ \left(-\frac{k_{2}\,{\theta _{2}}^2\,\left({a_{2}}^2\,k_{1}\,{\theta _{3}}^2-a_{2}\,b\,\theta _{1}\,\theta _{3}+b^2\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^2-a_{2}\,b\,k_{1}\,\theta _{1}\,\theta _{3}+2\,a_{2}\,b\,k_{1}\,k_{2}\,\theta _{1}\,\theta _{3}\right)}{b\,\theta _{1}\,{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^3\,{\left(k_{1}+1\right)}^3}\right)\,{a_{1}}^2, \\
        %
        Y_3 &= \frac{a_{2}\,\theta _{2}\,\theta _{3}}{b^2\,\theta _{1}\,{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^3\,{\left(k_{1}+1\right)}^3} \left({a_{2}}^2\,b\,{k_{1}}^2\,\theta _{1}\,{\theta _{3}}^2+2\,{a_{2}}^2\,b\,k_{1}\,\theta_{1}\,{\theta _{3}}^2+{a_{2}}^2\,b\,\theta _{1}\,{\theta _{3}}^2 +b^3\,{k_{2}}^2\,{\theta _{1}}^3 \right.\\
        &+ \left. 2\,a_{2}\,b^2\,{k_{1}}^2\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+4\,a_{2}\,b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+2\,a_{2}\,b^2\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+b^3\,{k_{1}}^2\,{k_{2}}^2\,{\theta _{1}}^3+2\,b^3\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^3\right) \,a_{1} \\
        &+ \left(-\frac{a_{2}\,\theta _{2}\,\theta _{3}\,\left({a_{2}}^2\,k_{1}\,\theta _{2}\,{\theta _{3}}^2+b^2\,k_{2}\,{\theta _{1}}^2\,\theta _{2}+b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,\theta _{2}+b^2\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^2\,\theta _{2}+2\,a_{2}\,b\,k_{1}\,k_{2}\,\theta _{1}\,\theta _{2}\,\theta _{3}\right)}{b^2\,\theta _{1}\,{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^3\,{\left(k_{1}+1\right)}^3}\right)\,{a_{1}}^2. \\
    \end{split}
\end{equation} 

Hence the growth rate function is 
\begin{equation}
    \begin{split}
        &\overline{\lambda}(\theta_{1}, \theta_{2}, \theta_{3}) \\
        &=\frac{a_{2}\,\theta _{2}\,\theta _{3}}{b^2\,\theta _{1}\,{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^3\,{\left(k_{1}+1\right)}^3} \\
        & \times\left({a_{2}}^2\,b\,{k_{1}}^2\,\theta _{1}\,{\theta _{3}}^2+2\,{a_{2}}^2\,b\,k_{1}\,\theta_{1}\,{\theta _{3}}^2+{a_{2}}^2\,b\,\theta _{1}\,{\theta _{3}}^2 + 2\,a_{2}\,b^2\,{k_{1}}^2\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+4\,a_{2}\,b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,\theta _{3} \right. \\
        &+ \left. 2\,a_{2}\,b^2\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+b^3\,{k_{1}}^2\,{k_{2}}^2\,{\theta _{1}}^3+2\,b^3\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^3+b^3\,{k_{2}}^2\,{\theta _{1}}^3\right) \,a_{1} \\
        &+ \left(-\frac{a_{2}\,\theta _{2}\,\theta _{3}\,\left({a_{2}}^2\,k_{1}\,\theta _{2}\,{\theta _{3}}^2+b^2\,k_{2}\,{\theta _{1}}^2\,\theta _{2}+b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,\theta _{2}+b^2\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^2\,\theta _{2}+2\,a_{2}\,b\,k_{1}\,k_{2}\,\theta _{1}\,\theta _{2}\,\theta _{3}\right)}{b^2\,\theta _{1}\,{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^3\,{\left(k_{1}+1\right)}^3}\right)\,{a_{1}}^2.
    \end{split}
\end{equation}

By the method of Lagrange multipliers, set $ L(\theta) = \lambda(\theta) - \mu \left(\theta_{1} + \theta_{2} + \theta_{3}\right) $, where $ \mu $ is the Lagrange multiplier. The optimality conditions are given by
\begin{subequations}
    \begin{align}
        \frac{\partial L}{\partial \theta_{1}} &= - \frac{a_{2}\,\theta _{2}\,\theta _{3}}{b\,{\theta _{1}}^2\,{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^4\,{\left(k_{1}+1\right)}^3} \,\left(b^4\,{k_{2}}^3\,{\theta _{1}}^4 + 2\,{a_{2}}^2\,b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,{\theta _{3}}^2+{a_{2}}^2\,b^2\,k_{2}\,{\theta _{1}}^2\,{\theta _{3}}^2 \right. \\ 
        &+ 2\,a_{2}\,b^3\,{k_{1}}^2\,{k_{2}}^2\,{\theta _{1}}^3\,\theta _{3}+4\,a_{2}\,b^3\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^3\,\theta _{3}+2\,a_{2}\,b^3\,{k_{2}}^2\,{\theta _{1}}^3\,\theta _{3}+b^4\,{k_{1}}^2\,{k_{2}}^3\,{\theta _{1}}^4+2\,b^4\,k_{1}\,{k_{2}}^3\,{\theta _{1}}^4 \\
        &+ \left.{a_{2}}^2\,b^2\,{k_{1}}^2\,k_{2}\,{\theta _{1}}^2\,{\theta _{3}}^2\right)\,a_{1} \\
        &+ \frac{a_{2}\,\theta _{2}\,\theta _{3}}{b\,{\theta _{1}}^2\,{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^4\,\left(k_{1}+1\right)^3} \,\left(2\,b^3\,{k_{2}}^2\,{\theta _{1}}^3\,\theta _{2}+{a_{2}}^3\,k_{1}\,\theta _{2}\,{\theta _{3}}^3 \right. \\
        &+ 2\,b^3\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^3\,\theta _{2}+2\,b^3\,k_{1}\,{k_{2}}^3\,{\theta _{1}}^3\,\theta _{2}-a_{2}\,b^2\,k_{2}\,{\theta _{1}}^2\,\theta _{2}\,\theta _{3}-a_{2}\,b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,\theta _{2}\,\theta _{3} \\
        &+ \left. 4\,{a_{2}}^2\,b\,k_{1}\,k_{2}\,\theta _{1}\,\theta _{2}\,{\theta _{3}}^2+5\,a_{2}\,b^2\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^2\,\theta _{2}\,\theta _{3}\right)\,{a_{1}}^2 - \mu = 0,\\
        \frac{\partial L}{\partial \theta_{2}} &= \frac{a_{2}\,\theta _{3}\,\left({a_{2}}^2\,b\,{k_{1}}^2\,\theta _{1}\,{\theta _{3}}^2+2\,{a_{2}}^2\,b\,k_{1}\,\theta _{1}\,{\theta _{3}}^2+{a_{2}}^2\,b\,\theta _{1}\,{\theta _{3}}^2+2\,a_{2}\,b^2\,{k_{1}}^2\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+4\,a_{2}\,b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+2\,a_{2}\,b^2\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+b^3\,{k_{1}}^2\,{k_{2}}^2\,{\theta _{1}}^3+2\,b^3\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^3+b^3\,{k_{2}}^2\,{\theta _{1}}^3\right)}{b\,\theta _{1}\,{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^3\,{\left(k_{1}+1\right)}^3}\,a_{1} \\
        &-\frac{a_{2}\,\theta _{3}\,\left(2\,{a_{2}}^2\,k_{1}\,\theta _{2}\,{\theta _{3}}^2+2\,b^2\,k_{2}\,{\theta _{1}}^2\,\theta _{2}+2\,b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,\theta _{2}+2\,b^2\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^2\,\theta _{2}+4\,a_{2}\,b\,k_{1}\,k_{2}\,\theta _{1}\,\theta _{2}\,\theta _{3}\right)}{b\,\theta _{1}\,{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^3\,{\left(k_{1}+1\right)}^3}\,{a_{1}}^2 \\
        \frac{\partial L}{\partial \theta_{3}} &= \frac{a_{2}\,k_{2}\,\theta _{2}\,\left({a_{2}}^2\,b\,{k_{1}}^2\,\theta _{1}\,{\theta _{3}}^2+2\,{a_{2}}^2\,b\,k_{1}\,\theta _{1}\,{\theta _{3}}^2+{a_{2}}^2\,b\,\theta _{1}\,{\theta _{3}}^2+2\,a_{2}\,b^2\,{k_{1}}^2\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+4\,a_{2}\,b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+2\,a_{2}\,b^2\,k_{2}\,{\theta _{1}}^2\,\theta _{3}+b^3\,{k_{1}}^2\,{k_{2}}^2\,{\theta _{1}}^3+2\,b^3\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^3+b^3\,{k_{2}}^2\,{\theta _{1}}^3\right)}{{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^4\,{\left(k_{1}+1\right)}^3}\,a_{1} \\
        &-\frac{a_{2}\,k_{2}\,\theta _{2}\,\left({a_{2}}^2\,k_{1}\,\theta _{2}\,{\theta _{3}}^2+b^2\,k_{2}\,{\theta _{1}}^2\,\theta _{2}+b^2\,k_{1}\,k_{2}\,{\theta _{1}}^2\,\theta _{2}-2\,a_{2}\,b\,\theta _{1}\,\theta _{2}\,\theta _{3}+b^2\,k_{1}\,{k_{2}}^2\,{\theta _{1}}^2\,\theta _{2}-2\,a_{2}\,b\,k_{1}\,\theta _{1}\,\theta _{2}\,\theta _{3}+2\,a_{2}\,b\,k_{1}\,k_{2}\,\theta _{1}\,\theta _{2}\,\theta _{3}\right)}{{\left(a_{2}\,\theta _{3}+b\,k_{2}\,\theta _{1}\right)}^4\,{\left(k_{1}+1\right)}^3}\,{a_{1}}^2 - \mu = 0.
    \end{align}
\end{subequations}


\subsubsection{General Discussion}
It may be expected that various bottleneck reactions in a long metabolic chain can be treated similarly, i.e. the steady-state protein fraction distribution given $ a_i $ is the single bottleneck reaction constant should be similar. However, the mathematical treatment returns vastly different forms analytically.

For simplicity, we will discuss the starvation and overabundance limits with a single bottleneck reaction, leading to more analytic details. The general case with multiple bottlenecks has been treated above with extensive approximation, so much physics has been masked.

\section{Theory for Mass-Action Kinetics}
\subsection{Solution of the Three-Sector Partition Model}
The three-sector model with mass-action kinetics, unlike the more general Michaelis-Menten kinetics discussed above, can be solved analytically. The system is described by the system of ODEs:
\begin{subequations}
    \begin{align}
        \frac{\mathrm{d}Y_1}{\mathrm{d}t} &= b Y_3 - r_1 Y_1 Y_4 - b Y_1 Y_3, \\
        \frac{\mathrm{d}Y_2}{\mathrm{d}t} &= r_1 Y_1 Y_4 - r_2 Y_2 Y_5 - b Y_2 Y_3, \\
        \frac{\mathrm{d}Y_3}{\mathrm{d}t} &= \theta_1 r_2 Y_2 Y_5 - b Y_3^2, \\
        \frac{\mathrm{d}Y_4}{\mathrm{d}t} &= \theta_2 r_2 Y_2 Y_5 - b Y_3 Y_4, \\
        \frac{\mathrm{d}Y_5}{\mathrm{d}t} &= \theta_3 r_2 Y_2 Y_5 - b Y_3 Y_5.
    \end{align}
\end{subequations}

The steady-state solutions are obtained by setting $ \mathrm{d}Y / \mathrm{d}t \to 0 $, and suppress the $ ^\ast $ for optimized values. 
\begin{subequations}
    \begin{align}
        b Y_3 &= r_1 Y_1 Y_4 + b Y_1 Y_3, \\
        r_1 Y_1 Y_4 &= r_2 Y_2 Y_5 + b Y_2 Y_3, \\
        \theta_1 r_2 Y_2 Y_5 &= b Y_3^2, \\
        \theta_2 r_2 Y_2 Y_5 &= b Y_3 Y_4, \\
        \theta_3 r_2 Y_2 Y_5 &= b Y_3 Y_5.
    \end{align}
\end{subequations}
Solving them, again with $ Y_3 \,:\, Y_4 \,:\, Y_5 = \theta_1 \,:\, \theta_2 \,:\, \theta_3 $, gives
\begin{subequations}
    \begin{align}
        Y_1 &= \left( 1 + \frac{r_1 \theta_{2}}{b \theta_{1}} \right)^{-1}, \\
        Y_2 &= \left(\frac{r_1 \theta_{2}}{b \theta_{1}}\right) \left[ \left(1 + \frac{r_2 \theta_{3}}{b \theta_{1}} \right)\left( 1 + \frac{r_1 \theta_{2}}{b \theta_{1}} \right) \right] ^{-1}, \\
        Y_3 &= \left(\frac{r_{2}\theta_{3}}{b}Y_2 \right), \quad Y_4 = \left(\frac{r_2 \theta_2 \theta_3 }{b \theta_1} \right) Y_2, \quad Y_5 = \left(\frac{r_2 \theta_3^2 }{b \theta_1} \right) Y_2. \\
    \end{align}
\end{subequations}

Interestingly, in this limit, the partition strength $ \theta $ has an analytic solution in the form of a root to a sixth-degree polynomial. The partition strength corresponding to ribosome fraction $ \theta_3 $ is a root of the polynomial 
\begin{equation}
    \begin{split}
        &\left( r_1^2 + 2r_1r_2 + r_2^2 - 5r_1^2r_2 - 2r_1r_2^2 - r_2^3 - r_2^4 + 4r_1^3r_2^2 + 4r_1^2r_2^3 + 3r_1r_2^4 - 4r_1^3r_2^3 - 3r_1^2r_2^4 + r_1^3r_2^4 \right) x^6 \\
        &- \left( 5r_1^2 + 2r_1r_2 + r_2^2 - 8r_1^2r_2 + r_1r_2^2 + 3r_2^3 - 4r_1^3r_2 - 11r_1^2r_2^2 - 7r_1r_2^3 + 14r_1^3r_2^2 + 10r_1^2r_2^3 - 6r_1^3r_2^3 \right) x^5 \\
        &+ \left( 8r_1^2 - r_1r_2 - 2r_2^2 + r_1^3 + 7r_1^2r_2 + 6r_1r_2^2 - 14r_1^3r_2 - 22r_1^2r_2^2 - 2r_1r_2^3 + 17r_1^3r_2^2 + 4r_1^2r_2^3 - 2r_1^3r_2^3 \right) x^4 \\
        &- r_1 \left( 2r_1 - r_2 + 4r_1^2 + 22r_1r_2 + 3r_2^2 - 18r_1^2r_2 - 13r_1r_2^2 + 8r_1^2r_2^2 \right) x^3 \\
        &+ r_1 \left( -7r_1 - r_2 + 6r_1^2 + 16r_1r_2 + r_2^2 - 10r_1^2r_2 - 2r_1r_2^2 + r_1^2r_2^2 \right) x^2 \\
        &+ r_1 \left( 7r_1 + r_2 - 4r_1^2 - 4r_2r_1 + 2r_2r_1^2 \right) x + r_1^2 \left( r_1 - 2 \right) ,
    \end{split}
\end{equation}
where we have set $ b=1 $ by a change of time unit. The other two partition strengths are fifth-degree polynomials in $ \theta_3 $. Checking with MATLAB \verb|factor()| returns no further factorization. 

To get a better understanding of the behavior of the partition strengths, we consider various limits of the system, including \textbf{starvation}, \textbf{overabundance of nutrient}, and the presence of a \textbf{single bottleneck reaction}. Later, we will extend the analysis to $ m $ bottleneck reactions in a general metabolic chain with $ N $ partitions.

For each limit, we define the growth rate objective function $ L(\theta, \mu) = bY_3(\theta) - \mu \left(\theta_1 + \theta_2 + \theta_3 - 1\right)$.

\subsection{Derivation of Partition Strengths in the Starvation Limit}
Again we expand the protein fractions as a series in $ b $. 
\begin{subequations}
    \begin{align}
        Y_1 &= \frac{\theta _{1}}{r_{1}\,\theta _{2}}\,b - \frac{{\theta _{1}}^2}{{r_{1}}^2\,{\theta _{2}}^2}\,b^2 + O(b^3), \\
        Y_2 &= \frac{\theta _{1}}{r_{2}\,\theta _{3}}\,b -\frac{\theta _{1}\,\left(r_{1}\,\theta _{1}\,\theta _{2}+r_{2}\,\theta _{1}\,\theta _{3}\right)}{r_{1}\,{r_{2}}^2\,\theta _{2}\,{\theta _{3}}^2}\,b^2 + O(b^3), \\
        Y_3 &= \theta _{1} -\frac{\theta _{1}\,\left(\theta _{1}\,{r_{1}}^2\,r_{2}\,{\theta _{2}}^2\,\theta _{3}+\theta _{1}\,r_{1}\,{r_{2}}^2\,\theta _{2}\,{\theta _{3}}^2\right)}{{r_{1}}^2\,{r_{2}}^2\,{\theta _{2}}^2\,{\theta _{3}}^2}\,b \\
        &+ \frac{\theta _{1}\,\left({r_{1}}^2\,{\theta _{1}}^2\,{\theta _{2}}^2+r_{1}\,r_{2}\,{\theta _{1}}^2\,\theta _{2}\,\theta _{3}+{r_{2}}^2\,{\theta _{1}}^2\,{\theta _{3}}^2\right)}{{r_{1}}^2\,{r_{2}}^2\,{\theta _{2}}^2\,{\theta _{3}}^2}\,b^2 + O(b^3).
    \end{align}
\end{subequations}

Lagrange multipliers can be used to solve for the optimal $ \theta $ values in this limit. 
\begin{subequations}
    \begin{align}
        \frac{\partial L}{\partial \theta_1} &= b -\frac{2\,\theta _{1}\,{r_{1}}^2\,r_{2}\,{\theta _{2}}^2\,\theta _{3}+2\,\theta _{1}\,r_{1}\,{r_{2}}^2\,\theta _{2}\,{\theta _{3}}^2}{{r_{1}}^2\,{r_{2}}^2\,{\theta _{2}}^2\,{\theta _{3}}^2}\,b^2 \\
        &+ \frac{3\,{r_{1}}^2\,{\theta _{1}}^2\,{\theta _{2}}^2+3\,r_{1}\,r_{2}\,{\theta _{1}}^2\,\theta _{2}\,\theta _{3}+3\,{r_{2}}^2\,{\theta _{1}}^2\,{\theta _{3}}^2}{{r_{1}}^2\,{r_{2}}^2\,{\theta _{2}}^2\,{\theta _{3}}^2}\,b^3 + O(b^4) - \mu = 0, \\
        \frac{\partial L}{\partial \theta_2} &= \frac{{\theta _{1}}^2}{r_{1}\,{\theta _{2}}^2}\,b^2 - \frac{{\theta _{1}}^2\,\left(r_{1}\,\theta _{1}\,\theta _{2}+2\,r_{2}\,\theta _{1}\,\theta _{3}\right)}{{r_{1}}^2\,r_{2}\,{\theta _{2}}^3\,\theta _{3}}\,b^3 + O(b^4) - \mu = 0, \\
        \frac{\partial L}{\partial \theta_3} &= \frac{{\theta _{1}}^2}{r_{2}\,{\theta _{3}}^2}\,b^2 - \frac{{\theta _{1}}^2\,\left(2\,r_{1}\,\theta _{1}\,\theta _{2}+r_{2}\,\theta _{1}\,\theta _{3}\right)}{r_{1}\,{r_{2}}^2\,\theta _{2}\,{\theta _{3}}^3}\,b^3 + O(b^4) - \mu = 0.
    \end{align}
\end{subequations}

\subsection{Derivation of Partition Strengths in the Overabundance Limit}
We expand the protein fractions as a series in $ 1/b $. 
\begin{subequations}
    \begin{align}
        Y_1 &= 1 - \frac{r_1\, \theta_2}{\theta_1 \,b} + \frac{r_1^2 \,\theta_2^2}{\theta_1^2 \, b^2} + O(b^{-3}), \\
        Y_2 &= \frac{r_1 \,\theta_2}{\theta_1\, b} - \frac{r_1^2\, \theta_2^2 + r_1\,r_2\,\theta_2\,\theta_3}{\theta_1^2\, b^2} + O(b^{-3}) , \\
        Y_3 &= \frac{r_{1}\,r_{2}\,\theta _{2}\,\theta _{3}}{\theta _{1}\,b^2} + O(b^{-3}) .
    \end{align}
\end{subequations}

Lagrange multipliers can be used to solve for the optimal $ \theta $ values in this limit. Since the leading term is already $ O(b^{-3}) $, we keep terms up to order $ b^{-4} $. 
\begin{subequations}
    \begin{align}
        \frac{\partial Y_1}{\partial \theta_1} &= \frac{r_{1}\,r_{2}\,\theta _{2}\,\theta _{3}\,\left(2\,r_{1}\,\theta _{2}-b\,\theta _{1}+2\,r_{2}\,\theta _{3}\right)}{b^2\,{\theta _{1}}^3} + O(b^{-3}) - \mu = 0, \\
        \frac{\partial Y_2}{\partial \theta_2} &= -\frac{r_{1}\,r_{2}\,\theta _{3}\,\left(2\,r_{1}\,\theta _{2}-b\,\theta _{1}+r_{2}\,\theta _{3}\right)}{b^2\,{\theta _{1}}^2} + O(b^{-3}) - \mu = 0, \\
        \frac{\partial Y_3}{\partial \theta_3} &= -\frac{r_{1}\,r_{2}\,\theta _{2}\,\left(r_{1}\,\theta _{2}-b\,\theta _{1}+2\,r_{2}\,\theta _{3}\right)}{b^2\,{\theta _{1}}^2} + O(b^{-3}) - \mu 0.
    \end{align}
\end{subequations}

\subsection{Derivation of Partition Strengths with Bottleneck Reactions}
We first expand the protein fractions as a series in $ r_1 $. 
\begin{subequations}
    \begin{align}
        Y_1 &= 1 - \left(\frac{\theta _{2}}{b\,\theta _{1}}\right)\,r_{1} + \frac{{\theta _{2}}^2}{b^2\,{\theta _{1}}^2}\,{r_{1}}^2 + O(r_1^3) , \\
        Y_2 &= \frac{\theta _{2}}{b\,\theta _{1}+r_{2}\,\theta _{3}}\,r_{1} - \frac{{\theta _{2}}^2}{b\,\theta _{1}\,\left(b\,\theta _{1}+r_{2}\,\theta _{3}\right)}\,{r_{1}}^{2} + O(r_1^3) , \\
        Y_3 &= \frac{r_{2}\,\theta _{2}\,\theta _{3}}{b\,\left(b\,\theta _{1}+r_{2}\,\theta _{3}\right)}\,r_{1} - \frac{r_{2}\,{\theta _{2}}^2\,\theta _{3}}{b^2\,\theta _{1}\,\left(b\,\theta _{1}+r_{2}\,\theta _{3}\right)}\,{r_{1}}^2 + O(r_1^3) .
    \end{align}
\end{subequations}

Lagrange multipliers can be used to solve for the optimal $ \theta $ values in this limit. 
\begin{subequations}
    \begin{align}
        \frac{\partial Y_1}{\partial \theta_1} &= \frac{b\,r_{2}\,\theta _{2}\,\theta _{3}}{{\left(b\,\theta _{1}+r_{2}\,\theta _{3}\right)}^2}\,r_1 + \frac{r_{2}\,\theta _{2}\,\theta _{3}\,\left(2\,b\,\theta _{1}\,\theta _{2}+r_{2}\,\theta _{2}\,\theta _{3}\right)}{b\,{\theta _{1}}^2\,{\left(b\,\theta _{1}+r_{2}\,\theta _{3}\right)}^2}\,{r_{1}}^2 + O(r_1^3) - \mu = 0, \\
        \frac{\partial Y_2}{\partial \theta_2} &= \frac{r_{2}\,\theta _{3}}{b\,\theta _{1}+r_{2}\,\theta _{3}}\,r_{1} -\frac{2\,r_{2}\,\theta _{2}\,\theta _{3}}{b\,\theta _{1}\,\left(b\,\theta _{1}+r_{2}\,\theta _{3}\right)}\,{r_{1}}^2 + O(r_1^3) - \mu = 0, \\
        \frac{\partial Y_3}{\partial \theta_3} &= \frac{b\,r_{2}\,\theta _{1}\,\theta _{2}}{{\left(b\,\theta _{1}+r_{2}\,\theta _{3}\right)}^2}\,r_{1} - \frac{r_{2}\,{\theta _{2}}^2}{{\left(b\,\theta _{1}+r_{2}\,\theta _{3}\right)}^2}\,{r_{1}}^2 + O(r_1^3) - \mu =0.
    \end{align}
\end{subequations}

Similarly, we can expand the protein fractions as a series in $ r_2 $.
\begin{subequations}
    \begin{align}
        Y_1 &= \frac{b\,\theta _{1}}{b\,\theta _{1}+r_{1}\,\theta _{2}} + O(r_2^3), \\
        Y_2 &= \frac{r_{1}\,\theta _{2}}{b\,\theta _{1}+r_{1}\,\theta _{2}} -\frac{r_{1}\,\theta _{2}\,\theta _{3}}{b\,\theta _{1}\,\left(b\,\theta _{1}+r_{1}\,\theta _{2}\right)}\,r_{2} + \frac{r_{1}\,\theta _{2}\,{\theta _{3}}^2}{b^2\,{\theta _{1}}^2\,\left(b\,\theta _{1}+r_{1}\,\theta _{2}\right)}\,{r_{2}}^2 + O(r_2^3), \\
        Y_3 &= \frac{r_{1}\,\theta _{2}\,\theta _{3}}{b\,\left(b\,\theta _{1}+r_{1}\,\theta _{2}\right)}\,r_{2} - \frac{r_{1}\,\theta _{2}\,{\theta _{3}}^2}{b^2\,\theta _{1}\,\left(b\,\theta _{1}+r_{1}\,\theta _{2}\right)}\,{r_{2}}^2 + O(r_2^3).
    \end{align}
\end{subequations}

Lagrange multipliers can be used to solve for the optimal $ \theta $ values in this limit. 
\begin{subequations}
    \begin{align}
        \frac{\partial Y_1}{\partial \theta_1} &= -\frac{b\,r_{1}\,\theta _{2}\,\theta _{3}}{{\left(b\,\theta _{1}+r_{1}\,\theta _{2}\right)}^2}\,r_{2} + \frac{r_{1}\,\theta _{2}\,\theta _{3}\,\left(2\,\theta _{3}\,b^2\,{\theta _{1}}^2+r_{1}\,\theta _{2}\,\theta _{3}\,b\,\theta _{1}\right)}{b^2\,{\theta _{1}}^3\,{\left(b\,\theta _{1}+r_{1}\,\theta _{2}\right)}^2}\,{r_{2}}^2 + O(r_2^3) - \mu = 0, \\
        \frac{\partial Y_2}{\partial \theta_2} &= \frac{b\,r_{1}\,\theta _{1}\,\theta _{3}}{{\left(b\,\theta _{1}+r_{1}\,\theta _{2}\right)}^2}\,r_{2} - \frac{r_{1}\,{\theta _{3}}^2}{{\left(b\,\theta _{1}+r_{1}\,\theta _{2}\right)}^2}\,{r_{2}}^2 + O(r_2^3) - \mu = 0, \\
        \frac{\partial Y_3}{\partial \theta_3} &= \frac{r_{1}\,\theta _{2}}{b\,\theta _{1}+r_{1}\,\theta _{2}}\,r_{2} - \frac{2\,r_{1}\,\theta _{2}\,\theta _{3}}{b\,\theta _{1}\,\left(b\,\theta _{1}+r_{1}\,\theta _{2}\right)}\,{r_{2}}^2 + O(r_2^3) - \mu = 0.
    \end{align}
\end{subequations}

\newpage
\section{Numerical Methods}
\subsection{Simulation Parameters}
Here is a detailed estimate of the average number of total molecules involved in the synthesis of a protein in an \textit{E. coli} cell. The molecules will include both small molecules such as metabolic intermediates and amino acids, and large molecules such as proteins and other polymeric compounds. 

For a ribosome to synthesize a protein, it requires the following ingredients: amino acids ($\approx 350$), tRNA ($\approx 350$), ATP/GTP $\approx 1050$, ribosomes ($\approx 350$). The average protein length in E. coli is 350 amino acids. Although ribosomes are not consumed, each protein needs one ribosome to synthesize. The above is summarized in table \ref{tab:protein_makeup}, and we find that on average about $2100$ molecules are involved (though not necessarily \textit{consumed}).

\begin{table}[ht]
\centering
\caption{Estimated counts of total molecules involved in the synthesis of a protein on average in a typical \textit{E. coli} Cell}
\begin{tabularx}{\textwidth}{
    >{\raggedright\arraybackslash}p{0.25\textwidth}
    >{\raggedright\arraybackslash}p{0.15\textwidth}
    >{\raggedright\arraybackslash}p{0.50\textwidth}
    }
    \hline
    \textbf{Molecule type} & \textbf{Count} & \textbf{Notes} \\
    \hline
    Amino acids ($ N_{\text{AA}} $) & $ \approx 350 $ & The average protein length is $ 350 $ amino acids, while the longest is a helicase encoded by the \textit{lhr} gene, comprising $1,538$ amino acids. \\
    \hline
    tRNA molecules ($ N_{\text{tRNA}} $)  & $\approx 350 $ & Each amino acid is delivered to the ribosome by a specific tRNA molecule. \\
    \hline
    ATP/GTP ($ N_{\text{ATP/GPT}} $) & $ \approx 1050 $  & \textbf{Amino acid activation}: Each amino acid is activated by an aminoacyl-tRNA synthetase, consuming 1 ATP per amino acid. \textbf{Translation steps}: Each elongation cycle consumes 2 GTP, respectively for the binding of aminoacyl-tRNA to the A site and the translocation step [\ref{ref:Jakubowsky}]. \\
    \hline
    Ribosomes ($ N_{\text{rib}} $) & $ \approx 350 $ & Ribosomes are not  consumed, but each protein needs one ribosome to synthesise. \\
    \hline 
    Total molecule count ($ N_{\text{mol}}) $ & $ \approx 2100 $ & \\ 
    \hline
    \bottomrule
\end{tabularx}
\label{tab:protein_makeup}
\end{table}

\subsection{Optimization}
We can solve for the $\theta$ value corresponding to optimal growth rate by numerical optimization using $4$th-order Runge-Kutta or with the analytic solution in a simplex. To speed up the parameter sweep, we use a softmax transformation of the following form to formulate it as an unconstrained problem,
\begin{equation}
    \theta_1 \to \frac{e^u}{1+e^u + e^v}, \; \theta_2 \to \frac{1}{1+e^u + e^v}, \; \theta_3 \to \frac{e^v}{1+e^u + e^v},
\end{equation}
and optimize over the region $(u, v)\in (-\infty, 0]^2$. The simulation speed is greatly enhanced with the python optimization package \verb|scipy.optimize import minimize|, compared to traditional two-stage mninimization. Parallel processing is used for different $b$ values by calling \verb|from joblib import Parallel, delayed|.

The difference between different methods is shown in figure \ref{fig:optimize}.
\begin{figure}
    \centering
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/heatmap.png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=0.9\linewidth]{Images/l (b=10.0).png}
        \caption{}
    \end{subfigure}
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=0.9\linewidth]{Images/l_b10_ODE.png}
        \caption{}
    \end{subfigure}
    \caption{(a) Unconstrained optimization in $(-\infty, 0]^2$ after SoftMax transformation. (b) Constrained optimization in simplex using the analytical solution, as in \cite{lin2025biomasstransferautocatalyticreaction}. (c) Numerical solution of the system of coupled ODEs using the python optimization package.}
    \label{fig:optimize}
\end{figure}

\subsection{Numerical Simulations}
Here are additional graphs that did not make it into the summary document. 

\subsubsection{Comparisons and Contrasts}

Expression using the approximate theory for the starvation and abundant nutrient limits are shown in figures \ref{fig:three_sectors_theta_low}, \ref{fig:three_sectors_theta_low_theory} and \ref{fig:three_sectors_theta_high}. The approximate expressions do not seem to match well, but the growth rate to nutrient level curve fits the theory well. See figure \ref{fig:compare_theory} for a comparison.
\begin{figure}
    \centering
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/theta1 vs b (low b n=3 theory).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/theta2 vs b (low b n=3 theory).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/theta3 vs b (low b n=3 theory).png}
        \caption{}
    \end{subfigure}
    \caption{The partition strength follows square root trend with changing nutrient level during starvation. (a) Transport proteins $\theta_1$ (b) Metabolic proteins $\theta_2$ (c) Ribosomal and translational proteins $\theta_3$. Here we compare the numerical result to theory.}
    \label{fig:three_sectors_theta_low_theory}
\end{figure}
\begin{figure}
    \centering
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/theta1 vs b (low b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/theta2 vs b (low b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/theta3 vs b (low b n=3).png}
        \caption{}
    \end{subfigure}
    \caption{Partition strength follows square root trend with changing nutrient level during starvation. (a) Transport proteins $\theta_1$ (b) Metabolic proteins $\theta_2$ (c) Ribosomal and translational proteins $\theta_3$.}
    \label{fig:three_sectors_theta_low}
\end{figure}
\begin{figure}
    \centering
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/theta1 vs b (high b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/theta2 vs b (high b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/theta3 vs b (high b n=3).png}
        \caption{}
    \end{subfigure}
    \caption{Partition strength during abundant nutrients. (a) Transport proteins $\theta_1$ (b) Metabolic proteins $\theta_2$ (c) Ribosomal and translational proteins $\theta_3$.}
    \label{fig:three_sectors_theta_high}
\end{figure}

\begin{figure}
    \centering
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs b (low b n=3 no theory).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs b (low b n=3 theory).png}
        \caption{}
    \end{subfigure}
    \caption{(a) Growth rate to nutrient level curve, without theory prediction. (b) Same curve, with theory prediction. }
    \label{fig:compare_theory}
\end{figure}

Trajectories (fractions) for the biomass associated with each node are shown and compared in figures \ref{fig:traj_low_b} and \ref{fig:traj_high_b}. The trends for $Y_j$ are different in the small and large $b$ limits, and we can summarize them as
\begin{equation}
    \text{low $b$: } Y_1 \nearrow \; Y_2 \nearrow \; Y_3 \searrow \; Y_4 \nearrow \; Y_5 \nearrow,
\end{equation}
\begin{equation}
    \text{high $b$: } Y_1 \searrow \; Y_2 \searrow \; Y_3 \searrow\; Y_4 \searrow \; Y_5 \searrow.
\end{equation}

\begin{figure}
    \centering
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y1 vs b (low b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y2 vs b (low b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y3 vs b (low b n=3).png}
        \caption{}
    \end{subfigure}
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y4 vs b (low b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y5 vs b (low b n=3).png}
        \caption{}
    \end{subfigure}%
    \caption{Trajectories for $Y_j$ during starvation. (a) Metabolites $Y_1$ (b) Metabolites $Y_2$ (c) Transport proteins $Y_3$ (d) Metabolic proteins $Y_4$ (e) Ribosomal and translational proteins $Y_5$.}
    \label{fig:traj_low_b}
\end{figure}
\begin{figure}
    \centering
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y1 vs b (high b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y2 vs b (high b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y3 vs b (high b n=3).png}
        \caption{}
    \end{subfigure}
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y4 vs b (high b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.33\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Y5 vs b (high b n=3).png}
        \caption{}
    \end{subfigure}%
    \caption{Trajectories for $Y_j$ during abundant nutrient supply. (a) Metabolites $Y_1$ (b) Metabolites $Y_2$ (c) Transport proteins $Y_3$ (d) Metabolic proteins $Y_4$ (e) Ribosomal and translational proteins $Y_5$.}
    \label{fig:traj_high_b}
\end{figure}

\subsubsection{Some Similarities}
The trends in the small and large molecule fraction remains similar for $b$ in the small and large limits. However, the fraction of large molecules is very high ($\sim 0.995$) when nutrient is low, while it is only moderately high ($\sim 0.675$) when nutrient is abundant. Refer to figure \ref{fig:three_sectors_small_large} for the graphs.
\begin{figure}
    \centering
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Large vs b (high b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Small vs b (high b n=3).png}
        \caption{}
    \end{subfigure}
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Large vs b (low b n=3).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/Small vs b (low b n=3).png}
        \caption{}
    \end{subfigure}
    \caption{The trends for Small and Large with respect to nutrient level remains similar in both limits, but . (a) (b) (c) (d)}
    \label{fig:three_sectors_small_large}
\end{figure}

\subsubsection{More Than Three Partitions}
The relationships between growth rate, nutrient level, and protein/molecule fractions remain the same in the starvation limit when we generalize to more partitions, figure \ref{fig:low_b_more_n}.

\begin{figure}
    \centering
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs b (low b n=6).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs Y6 (low b n=6).png}
        \caption{}
    \end{subfigure}
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs Y9 (low b n=4).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs Y6 (low b n=7).png}
        \caption{}
    \end{subfigure}
    \caption{Starvation. (a) Growth rate to $b$ relationship for six-sector model. This model has been considered in \cite{PRXLife.3.022001}, \cite{Chure_eLife} as a model for bacteria growth. (b) Growth rate vs $Y_6$ for six-sector model. (c) Ribosomal fraction $Y_9$ for four-sector model. (d) Growth rate vs housekeeping protein fraction $Y_7$ in six-sector model. }
    \label{fig:low_b_more_n}
\end{figure}

Differences emerge when we increase $b$. Also, $b$ in the high nutrient limit has qualitatively different behavior compared to the $n=3$ case. Figure \ref{fig:high_more_n}.

\begin{figure}
    \centering
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs theta (medium b n=6).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs theta (high b n=6) 02.png}
        \caption{}
    \end{subfigure}
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs Y5 (medium b n=6).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs Y6 (high b n=6).png}
        \caption{}
    \end{subfigure}
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs b (high b n=6).png}
        \caption{}
    \end{subfigure}%
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/lambda vs Y7 (high b n=6).png}
        \caption{}
    \end{subfigure}
    \caption{High or medium abundance of nutrients, with the six-sector model \cite{PRXLife.3.022001}. (a)  Growth rate vs $\theta$ for medium nutrient levels. (b) Growth rate vs $\theta$ for high nutrient levels. (c) Growth rate vs (one of the) metabolite(s) fraction. (d) Growth rate vs $Y_6$. (e) Growth rate vs $b$ relationship. (f) Growth rate vs one of the metabolic protein fractions. }
    \label{fig:high_more_n}
\end{figure}

\subsection{Correlation Between Biomass Fraction vs. Rate Constants}
We can also explore the correlation between biomass fractions $ Y_i $ and the corresponding rate constants $ a_i $ . Here we show some examples in figure \ref{fig:correlation_10}.

\begin{figure}
    \centering
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/dist_10 (b=0.01).png}
        \caption{Correlation between $Y_1$ and $a_1$ with $b=0.01$, where $ a_i $ are sorted.}
    \end{subfigure}%
    \begin{subfigure}[t]{0.5\linewidth}
        \centering
        \includegraphics[width=\linewidth]{Images/dist_10_unsorted (b=0.01).png}
        \caption{Correlation between $Y_1$ and $a_1$ with $b=10.0$, where $ a_i $ are unsorted and randomized.}
    \end{subfigure}
\end{figure}

\section{Appendix}
\subsection{Steady State Protein Fraction in the Overabundance Limit}
The steady-state protein fractions in the overabundance limit were given in symbolic form only, without explicitly giving the coefficients due to the complexity. Also, the expressions may not be very enlightening. Here we provide the full expressions.

The trajectories are given by 
\begin{align}
    Y_{1} &= 1 + \frac{\alpha^{2} a_{1}^{2} k_{1}}{(k_{1}+1)^{3}} - \frac{\alpha a_{1}}{k_{1}+1} - \frac{a_{1}\!\left(2k_{1}+k_{1}^{2}-2\alpha a_{1}k_{1}+1\right)}{\alpha(\alpha \beta+2)^{2}(k_{1}+1)^{3}b^2} + O(b^{-3}), \\
    %
    Y_{2} &= \frac{\alpha a_{1}}{k_{1}+1} - \frac{\alpha^{2} a_{1}^{2} \left(k_{1}+\dfrac{a_{2}(\alpha \beta+1)(k_{1}+1)^{2}}{a_{1}k_{2}}\right)}{(k_{1}+1)^{3}} \notag \\
    &- \frac{a_{1}\left(2\alpha a_{2}-k_{2}-2k_{1}k_{2}-k_{1}^{2}k_{2} +4\alpha a_{2}k_{1}+2\alpha^{2}\beta a_{2}+2\alpha a_{2}k_{1}^{2}+4\alpha^{2}\beta a_{2}k_{1} + 2\alpha^{2}\beta a_{2}k_{1}^{2}+2\alpha a_{1}k_{1}k_{2}\right)}{\alpha k_{2} (\alpha \beta+2)^{2}(k_{1}+1)^{3}b^2} \\
    &+ O(b^{-3}) \\
    &= \frac{\alpha a_{1}^2 \left[k_2 - \alpha(1+\alpha\beta)a_2\right]}{(1 + k_{1})k_{2}a_{1}} - \frac{\alpha^{2} k_{1} a_{1}^{2}}{(1 + k_{1})^{3}} + \frac{a_{1}\left[ \left( k_{2} - 2\alpha a_2 (1+\alpha\beta)\right)(1+k_1)^2 - 2\alpha k_1 k_2 a_1 \right]}{\alpha k_{2} (\alpha \beta+2)^{2}(k_{1}+1)^{3}b^2} + O(b^{-3}) , 
\end{align}
% Y3
\begin{align}
    Y_{3} 
    &= \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        4\alpha^{3}a_{1} + 4\alpha^{3}a_{2} - 4\alpha^{2}k_{2} + 4 \alpha^{3}a_{2}k_{1}^{2} - 4\alpha^{2}k_{1}^{2}k_{2} + 4\alpha^{4}\beta a_{1} \\
        + 8\alpha^{4}\beta a_{2} - 4\alpha^{3}\beta k_{2} + 4\alpha^{3}a_{1}k_{1} + 8\alpha^{3}a_{2}k_{1} - 8\alpha^{2}k_{1}k_{2} + \alpha^{5}\beta^{2}a_{1} + 5\alpha^{5}\beta^{2}a_{2} \\
        +\alpha^{6}\beta^{3}a_{2}  - \alpha^{4}\beta^{2}k_{2} - 8\alpha^{3}\beta k_{1}k_{2}  + 4\alpha^{3}a_{1}k_{1}k_{2} + 8\alpha^{4}\beta a_{2}k_{1}^{2} + \alpha^{5}\beta^{2}a_{1}k_{1} \\
        + 10\alpha^{5}\beta^{2}a_{2}k_{1} + 2\alpha^{6}\beta^{3}a_{2}k_{1} - 4\alpha^{3}\beta k_{1}^{2}k_{2} - 2\alpha^{4}\beta^{2}k_{1}k_{2} + 5\alpha^{5}\beta^{2}a_{2}k_{1}^{2} \\
        + \alpha^{6}\beta^{3}a_{2}k_{1}^{2} -\alpha^{4}\beta^{2}k_{1}^{2}k_{2} + 4\alpha^{4}\beta a_{1}k_{1} + 16\alpha^{4}\beta a_{2}k_{1} +4\alpha^{4}\beta a_{1}k_{1}k_{2} +\alpha^{5}\beta^{2}a_{1}k_{1}k_{2} )
    \end{gathered} \right)
    }{\alpha b k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \notag \\
    &+ \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        2\alpha k_{2} - 2\alpha^{2}a_{1} - 2\alpha^{2}a_{2} - \alpha^{3}\beta a_{1} - 3\alpha^{3}\beta a_{2} + \alpha^{2}\beta k_{2} - 2\alpha^{2}a_{1}k_{1} - 4\alpha^{2}a_{2} k_{1}\\
        + 2\alpha k_{1}^{2}k_{2} - \alpha^{4}\beta^{2}a_{2} - 2\alpha^{2}a_{2} k_{1}^{2} + \alpha^{2}\beta k_{1}^{2}k_{2} -\alpha^{4}\beta^{2}a_{2} k_{1}^{2}  - \alpha^{3}\beta a_{1}k_{1} - 6\alpha^{3}\beta a_{2} k_{1}\\
        + 2\alpha^{2}\beta  k_{1}k_{2} - 2\alpha^{2}a_{1} k_{1}k_{2} - 3\alpha^{3}\beta a_{2} k_{1}^{2} )
    \end{gathered} \right)
    }{\alpha b^{2} k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \notag \\
    &+ \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        3\alpha a_{1}-2k_{2} + 3\alpha a_{2} - 4k_{1}k_{2}-2k_{1}^{2}k_{2} + 3\alpha a_{1}k_{1} + 6\alpha a_{2}k_{1} + 3\alpha^{2}\beta a_{2} \\
        + 3\alpha a_{2}k_{1}^{2} + 6\alpha^{2}\beta a_{2}k_{1}
        + 3\alpha^{2}\beta a_{2}k_{1}^{2} + 3\alpha a_{1}k_{1}k_{2}+4\alpha b k_{1}k_{2} - 2\alpha^{4}\beta^{2}a_{2}b k_{1} - \alpha^{3}\beta a_{1}b k_{1}k_{2} )
    \end{gathered} \right)
    }{\alpha b^{3} k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \\
    &= \frac{G_3}{b} + \frac{H_3}{b^2} + O(b^{-3}),
\end{align}
% Y4
\begin{align}
    Y_{4} 
    &= \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        + 4\alpha^{3}a_{1} + 4\alpha^{3}a_{2} - 4\alpha^{2}k_{2} + 4\alpha^{3}a_{2}k_{1}^{2} - 4\alpha^{2}k_{1}^{2}k_{2} + 4\alpha^{4}\beta a_{1} + 8\alpha^{4}\beta a_{2} + 4\alpha^{3}a_{1}k_{1}\\
        + 8\alpha^{3}a_{2}k_{1} - 8\alpha^{2}k_{1}k_{2} + \alpha^{5}\beta^{2}a_{1} + 5\alpha^{5}\beta^{2}a_{2} + \alpha^{6}\beta^{3}a_{2} - \alpha^{4}\beta^{2}k_{2} + \alpha^{5}\beta^{2}a_{1}k_{1} + 10\alpha^{5}\beta^{2}a_{2}k_{1} \\
        + 2\alpha^{6}\beta^{3}a_{2}k_{1} - 4\alpha^{3}\beta k_{1}^{2}k_{2} - 2\alpha^{4}\beta^{2}k_{1}k_{2} - 4\alpha^{3}\beta k_{2} - 8\alpha^{3}\beta k_{1}k_{2} + 4\alpha^{3}a_{1}k_{1}k_{2} + 8\alpha^{4}\beta a_{2}k_{1}^{2} \\
        + 5\alpha^{5}\beta^{2}a_{2}k_{1}^{2} + \alpha^{6}\beta^{3}a_{2}k_{1}^{2} - \alpha^{4}\beta^{2}k_{1}^{2}k_{2} + 4\alpha^{4}\beta a_{1}k_{1} + 16\alpha^{4}\beta a_{2}k_{1} + 4\alpha^{4}\beta a_{1}k_{1}k_{2} + \alpha^{5}\beta^{2}a_{1}k_{1}k_{2})
        )
    \end{gathered} \right)
    }{\alpha k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \notag \\
    &+ \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        2\alpha k_{2} - 2\alpha^{2}a_{1} - 2\alpha^{2}a_{2} - \alpha^{3}\beta a_{1} - 3\alpha^{3}\beta a_{2} + \alpha^{2}\beta k_{2} - 2\alpha^{2}a_{1}k_{1} \\
        - 4\alpha^{2}a_{2}k_{1} + 2\alpha k_{1}^{2}k_{2} - 6\alpha^{3}\beta a_{2}k_{1} + 2\alpha^{2}\beta k_{1}k_{2} - 2\alpha^{2}a_{1}k_{1}k_{2} - \alpha^{4}\beta^{2}a_{2}k_{1}^{2} - \alpha^{3}\beta a_{1} k_{1} \\
        + 4\alpha k_{1}k_{2} + \alpha^{2}\beta k_{1}^{2}k_{2} - \alpha^{4}\beta^{2}a_{2} - 2\alpha^{2}a_{2}k_{1}^{2} - 3\alpha^{3}\beta a_{2}k_{1}^{2}  - 2\alpha^{4}\beta^{2}a_{2}k_{1} - \alpha^{3}\beta a_{1}k_{1}k_{2}
        )
    \end{gathered} \right)
    }{\alpha bk_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \notag \\
    &+ \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        4\alpha a_{1}-3k_{2} + 4\alpha a_{2}-6k_{1}k_{2} - 3k_{1}^{2}k_{2} + 4\alpha a_{1}k_{1} + 8\alpha a_{2}k_{1} + 4\alpha^{2}\beta a_{2} + 4\alpha a_{2}k_{1}^{2} \\
        + 8\alpha^{2}\beta a_{2}k_{1} + 4\alpha^{2}\beta a_{2}k_{1}^{2} + 4\alpha a_{1}k_{1}k_{2} 
        )
    \end{gathered} \right)
    }{\alpha b^{2} k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \\
    &= G_4 + \frac{H_4}{b} + O(b^{-2}),
\end{align}
% Y5
\begin{align}
    Y_{5}
    &= \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        4\alpha^{4}\beta a_{1}k_{1}k_{2} + \alpha^{5}\beta^{2}a_{1}k_{1}k_{2} + \alpha^{5}\beta^{2}a_{1}k_{1} + 10\alpha^{5}\beta^{2}a_{2}k_{1} + 2\alpha^{6}\beta^{3}a_{2}k_{1}\\
        - 4\alpha^{3}\beta k_{1}^{2}k_{2} - 2\alpha^{4}\beta^{2}k_{1}k_{2} - 8\alpha^{3}\beta k_{1}k_{2} + 4\alpha^{3}a_{1}k_{1}k_{2} + 8\alpha^{4}\beta a_{2}k_{1}^{2} + 4\alpha^{3}a_{1} + 4\alpha^{3}a_{2}\\
        - 4\alpha^{2}k_{2} + 4\alpha^{3}a_{2}k_{1}^{2} - 4\alpha^{2}k_{1}^{2}k_{2} + 4\alpha^{4}\beta a_{1} + 8\alpha^{4}\beta a_{2} + \alpha^{5}\beta^{2}a_{1} + 5\alpha^{5}\beta^{2}a_{2} + \alpha^{6}\beta^{3}a_{2}\\
        - \alpha^{4}\beta^{2}k_{2} + 5\alpha^{5}\beta^{2}a_{2}k_{1}^{2} + \alpha^{6}\beta^{3}a_{2}k_{1}^{2} -\alpha^{4}\beta^{2}k_{1}^{2}k_{2} + 4\alpha^{4}\beta a_{1}k_{1} + 16\alpha^{4}\beta a_{2}k_{1} + 4\alpha^{3}a_{1}k_{1}\\
        + 8\alpha^{3}a_{2}k_{1} - 8\alpha^{2}k_{1}k_{2} - 4\alpha^{3}\beta k_{2}
        )
    \end{gathered} \right)
    }{\alpha k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \notag \\
    &+ \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        \alpha^{2}\beta k_{2} - 2\alpha^{2}a_{1}k_{1} - 4\alpha^{2}a_{2}k_{1} + 2\alpha k_{1}^{2}k_{2} - 2\alpha^{2}a_{1} - 2\alpha^{2}a_{2} - \alpha^{3}\beta a_{1}\\
        - 3\alpha^{3}\beta a_{2} + \alpha^{2}\beta k_{1}^{2}k_{2} - \alpha^{4}\beta^{2}a_{2} k_{1}^{2} - \alpha^{3}\beta a_{1}k_{1} - 6\alpha^{3}\beta a_{2}k_{1} + 2\alpha^{2}\beta k_{1}k_{2} - 2\alpha^{2}a_{1}k_{1}k_{2} + 2\alpha k_{2}\\
        - \alpha^{4}\beta^{2}a_{2} + 4\alpha k_{1}k_{2} - 3\alpha^{3}\beta a_{2}k_{1}^{2} - 2\alpha^{4}\beta^{2}a_{2}k_{1} - \alpha^{3}\beta a_{1}k_{1}k_{2}
        )
    \end{gathered} \right)
    }{\alpha b k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \notag \\
    &+ \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        4\alpha a_{1} - 3k_{2} + 4\alpha a_{2} - 6 k_{1}k_{2} - 3 k_{1}^{2}k_{2} + 4\alpha a_{1}k_{1} + 8\alpha a_{2}k_{1} + 4\alpha^{2}\beta a_{2}\\
        + 4\alpha a_{2}k_{1}^{2} + 8\alpha^{2}\beta a_{2}k_{1} - 2\alpha^{2}a_{2}b k_{1}^{2} + 4\alpha^{2}\beta a_{2}k_{1}^{2} + 4\alpha a_{1}k_{1}k_{2}
        )
    \end{gathered} \right)
    }{\alpha b^{2} k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \\
    &= G_5 + \frac{H_5}{b} + O(b^{-2}),
\end{align}

Then the growth rate is
% lambda
\begin{align}
    \lambda &= bY_3 \\
    &= \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        4\alpha^{3}a_{1} + 4\alpha^{3}a_{2} - 4\alpha^{2}k_{2} + 4 \alpha^{3}a_{2}k_{1}^{2} - 4\alpha^{2}k_{1}^{2}k_{2} + 4\alpha^{4}\beta a_{1} \\
        + 8\alpha^{4}\beta a_{2} - 4\alpha^{3}\beta k_{2} + 4\alpha^{3}a_{1}k_{1} + 8\alpha^{3}a_{2}k_{1} - 8\alpha^{2}k_{1}k_{2} + \alpha^{5}\beta^{2}a_{1} + 5\alpha^{5}\beta^{2}a_{2} \\
        +\alpha^{6}\beta^{3}a_{2}  - \alpha^{4}\beta^{2}k_{2} - 8\alpha^{3}\beta k_{1}k_{2}  + 4\alpha^{3}a_{1}k_{1}k_{2} + 8\alpha^{4}\beta a_{2}k_{1}^{2} + \alpha^{5}\beta^{2}a_{1}k_{1} \\
        + 10\alpha^{5}\beta^{2}a_{2}k_{1} + 2\alpha^{6}\beta^{3}a_{2}k_{1} - 4\alpha^{3}\beta k_{1}^{2}k_{2} - 2\alpha^{4}\beta^{2}k_{1}k_{2} + 5\alpha^{5}\beta^{2}a_{2}k_{1}^{2} \\
        + \alpha^{6}\beta^{3}a_{2}k_{1}^{2} -\alpha^{4}\beta^{2}k_{1}^{2}k_{2} + 4\alpha^{4}\beta a_{1}k_{1} + 16\alpha^{4}\beta a_{2}k_{1} +4\alpha^{4}\beta a_{1}k_{1}k_{2} +\alpha^{5}\beta^{2}a_{1}k_{1}k_{2} )
    \end{gathered} \right)
    }{\alpha k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \notag \\
    &+ \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        2\alpha k_{2} - 2\alpha^{2}a_{1} - 2\alpha^{2}a_{2} - \alpha^{3}\beta a_{1} - 3\alpha^{3}\beta a_{2} + \alpha^{2}\beta k_{2} - 2\alpha^{2}a_{1}k_{1} - 4\alpha^{2}a_{2} k_{1}\\
        + 2\alpha k_{1}^{2}k_{2} - \alpha^{4}\beta^{2}a_{2} - 2\alpha^{2}a_{2} k_{1}^{2} + \alpha^{2}\beta k_{1}^{2}k_{2} -\alpha^{4}\beta^{2}a_{2} k_{1}^{2}  - \alpha^{3}\beta a_{1}k_{1} - 6\alpha^{3}\beta a_{2} k_{1}\\
        + 2\alpha^{2}\beta  k_{1}k_{2} - 2\alpha^{2}a_{1} k_{1}k_{2} - 3\alpha^{3}\beta a_{2} k_{1}^{2} )
    \end{gathered} \right)
    }{\alpha b k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \notag \\
    &+ \frac{ \left(
    \begin{gathered}
        a_{1}a_{2}(\alpha\beta+1) ( 
        3\alpha a_{1}-2k_{2} + 3\alpha a_{2} - 4k_{1}k_{2}-2k_{1}^{2}k_{2} + 3\alpha a_{1}k_{1} + 6\alpha a_{2}k_{1} + 3\alpha^{2}\beta a_{2} \\
        + 3\alpha a_{2}k_{1}^{2} + 6\alpha^{2}\beta a_{2}k_{1}
        + 3\alpha^{2}\beta a_{2}k_{1}^{2} + 3\alpha a_{1}k_{1}k_{2}+4\alpha b k_{1}k_{2} - 2\alpha^{4}\beta^{2}a_{2}b k_{1} - \alpha^{3}\beta a_{1}b k_{1}k_{2} )
    \end{gathered} \right)
    }{\alpha b^{2} k_{2}^{2} (\alpha\beta+2)^{3}(k_{1}+1)^{3}} \\
    &= \lambda_0 - \frac{\lambda_1}{b} + O(b^{-2}).
\end{align}

% Bibliography
\bibliography{references}

\end{document}